{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "2b562365",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'d:\\\\Projects\\\\trec-covid-ir-3\\\\notebooks'"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import os\n",
    "import warnings\n",
    "warnings.filterwarnings('ignore')\n",
    "\n",
    "__root__ = os.getcwd()\n",
    "__root__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "69f1dff4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "cord_uid",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "sha",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "source_x",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "title",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "doi",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pmcid",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pubmed_id",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "license",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "abstract",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "publish_time",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "authors",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "journal",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "mag_id",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "who_covidence_id",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "arxiv_id",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pdf_json_files",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "pmc_json_files",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "url",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "s2_id",
         "rawType": "float64",
         "type": "float"
        }
       ],
       "ref": "78d7dbb4-20b9-43f5-a480-751bf4d16679",
       "rows": [
        [
         "0",
         "ug7v899j",
         "d1aafb70c066a2068b02786f8929fd9c900897fb",
         "PMC",
         "Clinical features of culture-proven Mycoplasma pneumoniae infections at King Abdulaziz University Hospital, Jeddah, Saudi Arabia",
         "10.1186/1471-2334-1-6",
         "PMC35282",
         "11472636.0",
         "no-cc",
         "OBJECTIVE: This retrospective chart review describes the epidemiology and clinical features of 40 patients with culture-proven Mycoplasma pneumoniae infections at King Abdulaziz University Hospital, Jeddah, Saudi Arabia. METHODS: Patients with positive M. pneumoniae cultures from respiratory specimens from January 1997 through December 1998 were identified through the Microbiology records. Charts of patients were reviewed. RESULTS: 40 patients were identified, 33 (82.5%) of whom required admission. Most infections (92.5%) were community-acquired. The infection affected all age groups but was most common in infants (32.5%) and pre-school children (22.5%). It occurred year-round but was most common in the fall (35%) and spring (30%). More than three-quarters of patients (77.5%) had comorbidities. Twenty-four isolates (60%) were associated with pneumonia, 14 (35%) with upper respiratory tract infections, and 2 (5%) with bronchiolitis. Cough (82.5%), fever (75%), and malaise (58.8%) were the most common symptoms, and crepitations (60%), and wheezes (40%) were the most common signs. Most patients with pneumonia had crepitations (79.2%) but only 25% had bronchial breathing. Immunocompromised patients were more likely than non-immunocompromised patients to present with pneumonia (8/9 versus 16/31, P = 0.05). Of the 24 patients with pneumonia, 14 (58.3%) had uneventful recovery, 4 (16.7%) recovered following some complications, 3 (12.5%) died because of M pneumoniae infection, and 3 (12.5%) died due to underlying comorbidities. The 3 patients who died of M pneumoniae pneumonia had other comorbidities. CONCLUSION: our results were similar to published data except for the finding that infections were more common in infants and preschool children and that the mortality rate of pneumonia in patients with comorbidities was high.",
         "2001-07-04",
         "Madani, Tariq A; Al-Ghamdi, Aisha A",
         "BMC Infect Dis",
         null,
         null,
         null,
         "document_parses/pdf_json/d1aafb70c066a2068b02786f8929fd9c900897fb.json",
         "document_parses/pmc_json/PMC35282.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC35282/",
         null
        ],
        [
         "1",
         "02tnwd4m",
         "6b0567729c2143a66d737eb0a2f63f2dce2e5a7d",
         "PMC",
         "Nitric oxide: a pro-inflammatory mediator in lung disease?",
         "10.1186/rr14",
         "PMC59543",
         "11667967.0",
         "no-cc",
         "Inflammatory diseases of the respiratory tract are commonly associated with elevated production of nitric oxide (NO•) and increased indices of NO• -dependent oxidative stress. Although NO• is known to have anti-microbial, anti-inflammatory and anti-oxidant properties, various lines of evidence support the contribution of NO• to lung injury in several disease models. On the basis of biochemical evidence, it is often presumed that such NO• -dependent oxidations are due to the formation of the oxidant peroxynitrite, although alternative mechanisms involving the phagocyte-derived heme proteins myeloperoxidase and eosinophil peroxidase might be operative during conditions of inflammation. Because of the overwhelming literature on NO• generation and activities in the respiratory tract, it would be beyond the scope of this commentary to review this area comprehensively. Instead, it focuses on recent evidence and concepts of the presumed contribution of NO• to inflammatory diseases of the lung.",
         "2000-08-15",
         "Vliet, Albert van der; Eiserich, Jason P; Cross, Carroll E",
         "Respir Res",
         null,
         null,
         null,
         "document_parses/pdf_json/6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json",
         "document_parses/pmc_json/PMC59543.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC59543/",
         null
        ],
        [
         "2",
         "ejv2xln0",
         "06ced00a5fc04215949aa72528f2eeaae1d58927",
         "PMC",
         "Surfactant protein-D and pulmonary host defense",
         "10.1186/rr19",
         "PMC59549",
         "11667972.0",
         "no-cc",
         "Surfactant protein-D (SP-D) participates in the innate response to inhaled microorganisms and organic antigens, and contributes to immune and inflammatory regulation within the lung. SP-D is synthesized and secreted by alveolar and bronchiolar epithelial cells, but is also expressed by epithelial cells lining various exocrine ducts and the mucosa of the gastrointestinal and genitourinary tracts. SP-D, a collagenous calcium-dependent lectin (or collectin), binds to surface glycoconjugates expressed by a wide variety of microorganisms, and to oligosaccharides associated with the surface of various complex organic antigens. SP-D also specifically interacts with glycoconjugates and other molecules expressed on the surface of macrophages, neutrophils, and lymphocytes. In addition, SP-D binds to specific surfactant-associated lipids and can influence the organization of lipid mixtures containing phosphatidylinositol in vitro. Consistent with these diverse in vitro activities is the observation that SP-D-deficient transgenic mice show abnormal accumulations of surfactant lipids, and respond abnormally to challenge with respiratory viruses and bacterial lipopolysaccharides. The phenotype of macrophages isolated from the lungs of SP-D-deficient mice is altered, and there is circumstantial evidence that abnormal oxidant metabolism and/or increased metalloproteinase expression contributes to the development of emphysema. The expression of SP-D is increased in response to many forms of lung injury, and deficient accumulation of appropriately oligomerized SP-D might contribute to the pathogenesis of a variety of human lung diseases.",
         "2000-08-25",
         "Crouch, Erika C",
         "Respir Res",
         null,
         null,
         null,
         "document_parses/pdf_json/06ced00a5fc04215949aa72528f2eeaae1d58927.json",
         "document_parses/pmc_json/PMC59549.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC59549/",
         null
        ],
        [
         "3",
         "2b73a28n",
         "348055649b6b8cf2b9a376498df9bf41f7123605",
         "PMC",
         "Role of endothelin-1 in lung disease",
         "10.1186/rr44",
         "PMC59574",
         "11686871.0",
         "no-cc",
         "Endothelin-1 (ET-1) is a 21 amino acid peptide with diverse biological activity that has been implicated in numerous diseases. ET-1 is a potent mitogen regulator of smooth muscle tone, and inflammatory mediator that may play a key role in diseases of the airways, pulmonary circulation, and inflammatory lung diseases, both acute and chronic. This review will focus on the biology of ET-1 and its role in lung disease.",
         "2001-02-22",
         "Fagan, Karen A; McMurtry, Ivan F; Rodman, David M",
         "Respir Res",
         null,
         null,
         null,
         "document_parses/pdf_json/348055649b6b8cf2b9a376498df9bf41f7123605.json",
         "document_parses/pmc_json/PMC59574.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC59574/",
         null
        ],
        [
         "4",
         "9785vg6d",
         "5f48792a5fa08bed9f56016f4981ae2ca6031b32",
         "PMC",
         "Gene expression in epithelial cells in response to pneumovirus infection",
         "10.1186/rr61",
         "PMC59580",
         "11686888.0",
         "no-cc",
         "Respiratory syncytial virus (RSV) and pneumonia virus of mice (PVM) are viruses of the family Paramyxoviridae, subfamily pneumovirus, which cause clinically important respiratory infections in humans and rodents, respectively. The respiratory epithelial target cells respond to viral infection with specific alterations in gene expression, including production of chemoattractant cytokines, adhesion molecules, elements that are related to the apoptosis response, and others that remain incompletely understood. Here we review our current understanding of these mucosal responses and discuss several genomic approaches, including differential display reverse transcription-polymerase chain reaction (PCR) and gene array strategies, that will permit us to unravel the nature of these responses in a more complete and systematic manner.",
         "2001-05-11",
         "Domachowske, Joseph B; Bonville, Cynthia A; Rosenberg, Helene F",
         "Respir Res",
         null,
         null,
         null,
         "document_parses/pdf_json/5f48792a5fa08bed9f56016f4981ae2ca6031b32.json",
         "document_parses/pmc_json/PMC59580.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC59580/",
         null
        ],
        [
         "5",
         "zjufx4fo",
         "b2897e1277f56641193a6db73825f707eed3e4c9",
         "PMC",
         "Sequence requirements for RNA strand transfer during nidovirus discontinuous subgenomic RNA synthesis",
         "10.1093/emboj/20.24.7220",
         "PMC125340",
         "11742998.0",
         "green-oa",
         "Nidovirus subgenomic mRNAs contain a leader sequence derived from the 5′ end of the genome fused to different sequences (‘bodies’) derived from the 3′ end. Their generation involves a unique mechanism of discontinuous subgenomic RNA synthesis that resembles copy-choice RNA recombination. During this process, the nascent RNA strand is transferred from one site in the template to another, during either plus or minus strand synthesis, to yield subgenomic RNA molecules. Central to this process are transcription-regulating sequences (TRSs), which are present at both template sites and ensure the fidelity of strand transfer. Here we present results of a comprehensive co-variation mutagenesis study of equine arteritis virus TRSs, demonstrating that discontinuous RNA synthesis depends not only on base pairing between sense leader TRS and antisense body TRS, but also on the primary sequence of the body TRS. While the leader TRS merely plays a targeting role for strand transfer, the body TRS fulfils multiple functions. The sequences of mRNA leader–body junctions of TRS mutants strongly suggested that the discontinuous step occurs during minus strand synthesis.",
         "2001-12-17",
         "Pasternak, Alexander O.; van den Born, Erwin; Spaan, Willy J.M.; Snijder, Eric J.",
         "The EMBO Journal",
         null,
         null,
         null,
         "document_parses/pdf_json/b2897e1277f56641193a6db73825f707eed3e4c9.json",
         "document_parses/pmc_json/PMC125340.xml.json",
         "http://europepmc.org/articles/pmc125340?pdf=render",
         null
        ],
        [
         "6",
         "ymceytj3",
         "e3d0d482ebd9a8ba81c254cc433f314142e72174",
         "PMC",
         "Crystal structure of murine sCEACAM1a[1,4]: a coronavirus receptor in the CEA family",
         "10.1093/emboj/21.9.2076",
         "PMC125375",
         "11980704.0",
         "green-oa",
         "CEACAM1 is a member of the carcinoembryonic antigen (CEA) family. Isoforms of murine CEACAM1 serve as receptors for mouse hepatitis virus (MHV), a murine coronavirus. Here we report the crystal structure of soluble murine sCEACAM1a[1,4], which is composed of two Ig-like domains and has MHV neutralizing activity. Its N-terminal domain has a uniquely folded CC′ loop that encompasses key virus-binding residues. This is the first atomic structure of any member of the CEA family, and provides a prototypic architecture for functional exploration of CEA family members. We discuss the structural basis of virus receptor activities of murine CEACAM1 proteins, binding of Neisseria to human CEACAM1, and other homophilic and heterophilic interactions of CEA family members.",
         "2002-05-01",
         "Tan, Kemin; Zelus, Bruce D.; Meijers, Rob; Liu, Jin-huan; Bergelson, Jeffrey M.; Duke, Norma; Zhang, Rongguang; Joachimiak, Andrzej; Holmes, Kathryn V.; Wang, Jia-huai",
         "The EMBO Journal",
         null,
         null,
         null,
         "document_parses/pdf_json/e3d0d482ebd9a8ba81c254cc433f314142e72174.json",
         "document_parses/pmc_json/PMC125375.xml.json",
         "http://europepmc.org/articles/pmc125375?pdf=render",
         null
        ],
        [
         "7",
         "wzj2glte",
         "00b1d99e70f779eb4ede50059db469c65e8c1469",
         "PMC",
         "Synthesis of a novel hepatitis C virus protein by ribosomal frameshift",
         "10.1093/emboj/20.14.3840",
         "PMC125543",
         "11447125.0",
         "no-cc",
         "Hepatitis C virus (HCV) is an important human pathogen that affects ∼100 million people worldwide. Its RNA genome codes for a polyprotein, which is cleaved by viral and cellular proteases to produce at least 10 mature viral protein products. We report here the discovery of a novel HCV protein synthesized by ribosomal frameshift. This protein, which we named the F protein, is synthesized from the initiation codon of the polyprotein sequence followed by ribosomal frameshift into the −2/+1 reading frame. This ribosomal frameshift requires only codons 8–14 of the core protein‐coding sequence, and the shift junction is located at or near codon 11. An F protein analog synthesized in vitro reacted with the sera of HCV patients but not with the sera of hepatitis B patients, indicating the expression of the F protein during natural HCV infection. This unexpected finding may open new avenues for the development of anti‐HCV drugs.",
         "2001-07-16",
         "Xu, Zhenming; Choi, Jinah; Yen, T.S.Benedict; Lu, Wen; Strohecker, Anne; Govindarajan, Sugantha; Chien, David; Selby, Mark J.; Ou, Jing‐hsiung",
         "EMBO J",
         null,
         null,
         null,
         "document_parses/pdf_json/00b1d99e70f779eb4ede50059db469c65e8c1469.json",
         "document_parses/pmc_json/PMC125543.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC125543/",
         null
        ],
        [
         "8",
         "2sfqsfm1",
         "cf584e00f637cbd8f1bb35f3f09f5ed07b71aeb0",
         "PMC",
         "Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra α-helical domain",
         "10.1093/emboj/cdf327",
         "PMC126080",
         "12093723.0",
         "green-oa",
         "The key enzyme in coronavirus polyprotein processing is the viral main proteinase, M(pro), a protein with extremely low sequence similarity to other viral and cellular proteinases. Here, the crystal structure of the 33.1 kDa transmissible gastroenteritis (corona)virus M(pro) is reported. The structure was refined to 1.96 Å resolution and revealed three dimers in the asymmetric unit. The mutual arrangement of the protomers in each of the dimers suggests that M(pro) self-processing occurs in trans. The active site, comprised of Cys144 and His41, is part of a chymotrypsin-like fold that is connected by a 16 residue loop to an extra domain featuring a novel α-helical fold. Molecular modelling and mutagenesis data implicate the loop in substrate binding and elucidate S1 and S2 subsites suitable to accommodate the side chains of the P1 glutamine and P2 leucine residues of M(pro) substrates. Interactions involving the N-terminus and the α-helical domain stabilize the loop in the orientation required for trans-cleavage activity. The study illustrates that RNA viruses have evolved unprecedented variations of the classical chymotrypsin fold.",
         "2002-07-01",
         "Anand, Kanchan; Palm, Gottfried J.; Mesters, Jeroen R.; Siddell, Stuart G.; Ziebuhr, John; Hilgenfeld, Rolf",
         "The EMBO Journal",
         null,
         null,
         null,
         "document_parses/pdf_json/cf584e00f637cbd8f1bb35f3f09f5ed07b71aeb0.json",
         "document_parses/pmc_json/PMC126080.xml.json",
         "http://europepmc.org/articles/pmc126080?pdf=render",
         null
        ],
        [
         "9",
         "i0zym7iq",
         "dde02f11923815e6a16a31dd6298c46b109c5dfa",
         "PMC",
         "Discontinuous and non-discontinuous subgenomic RNA transcription in a nidovirus",
         "10.1093/emboj/cdf635",
         "PMC136939",
         "12456663.0",
         "green-oa",
         "Arteri-, corona-, toro- and roniviruses are evolutionarily related positive-strand RNA viruses, united in the order Nidovirales. The best studied nidoviruses, the corona- and arteriviruses, employ a unique transcription mechanism, which involves discontinuous RNA synthesis, a process resembling similarity-assisted copy-choice RNA recombination. During infection, multiple subgenomic (sg) mRNAs are transcribed from a mirror set of sg negative-strand RNA templates. The sg mRNAs all possess a short 5′ common leader sequence, derived from the 5′ end of the genomic RNA. The joining of the non-contiguous ‘leader’ and ‘body’ sequences presumably occurs during minus-strand synthesis. To study whether toroviruses use a similar transcription mechanism, we characterized the 5′ termini of the genome and the four sg mRNAs of Berne virus (BEV). We show that BEV mRNAs 3–5 lack a leader sequence. Surprisingly, however, RNA 2 does contain a leader, identical to the 5′-terminal 18 residues of the genome. Apparently, BEV combines discontinuous and non-discontinous RNA synthesis to produce its sg mRNAs. Our findings have important implications for the understanding of the mechanism and evolution of nidovirus transcription.",
         "2002-12-01",
         "van Vliet, A.L.W.; Smits, S.L.; Rottier, P.J.M.; de Groot, R.J.",
         "The EMBO Journal",
         null,
         null,
         null,
         "document_parses/pdf_json/dde02f11923815e6a16a31dd6298c46b109c5dfa.json",
         "document_parses/pmc_json/PMC136939.xml.json",
         "http://europepmc.org/articles/pmc136939?pdf=render",
         null
        ],
        [
         "10",
         "5yhe786e",
         "3bb07ea10432f7738413dff9816809cc90f03f99",
         "PMC",
         "Debate: Transfusing to normal haemoglobin levels will not improve outcome",
         "10.1186/cc987",
         "PMC137267",
         "11299062.0",
         "no-cc",
         "Recent evidence suggests that critically ill patients are able to tolerate lower levels of haemoglobin than was previously believed. It is our goal to show that transfusing to a level of 100 g/l does not improve mortality and other clinically important outcomes in a critical care setting. Although many questions remain, many laboratory and clinical studies, including a recent randomized controlled trial (RCT), have established that transfusing to normal haemoglobin concentrations does not improve organ failure and mortality in the critically ill patient. In addition, a restrictive transfusion strategy will reduce exposure to allogeneic transfusions, result in more efficient use of red blood cells (RBCs), save blood overall, and decrease health care costs.",
         "2001-03-08",
         "Alvarez, Gonzalo; Hébert, Paul C; Szick, Sharyn",
         "Crit Care",
         null,
         null,
         null,
         "document_parses/pdf_json/3bb07ea10432f7738413dff9816809cc90f03f99.json",
         "document_parses/pmc_json/PMC137267.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC137267/",
         null
        ],
        [
         "11",
         "8zchiykl",
         "5806726a24dc91de3954001effbdffd7a82d54e2",
         "PMC",
         "The 21st International Symposium on Intensive Care and Emergency Medicine, Brussels, Belgium, 20-23 March 2001",
         "10.1186/cc1013",
         "PMC137274",
         "11353930.0",
         "no-cc",
         "The 21st International Symposium on Intensive Care and Emergency Medicine was dominated by the results of recent clinical trials in sepsis and acute respiratory distress syndrome (ARDS). The promise of extracorporeal liver replacement therapy and noninvasive ventilation were other areas of interest. Ethical issues also received attention. Overall, the 'state of the art' lectures, pro/con debates, seminars and tutorials were of a high standard. The meeting was marked by a sense of renewed enthusiasm that positive progress is occurring in intensive care medicine.",
         "2001-05-02",
         "Ball, Jonathan; Venn, Richard",
         "Crit Care",
         null,
         null,
         null,
         "document_parses/pdf_json/5806726a24dc91de3954001effbdffd7a82d54e2.json",
         "document_parses/pmc_json/PMC137274.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC137274/",
         null
        ],
        [
         "12",
         "xqhn0vbp",
         "1e1286db212100993d03cc22374b624f7caee956",
         "PMC",
         "Airborne rhinovirus detection and effect of ultraviolet irradiation on detection by a semi-nested RT-PCR assay",
         "10.1186/1471-2458-3-5",
         "PMC140314",
         "12525263.0",
         "no-cc",
         "BACKGROUND: Rhinovirus, the most common cause of upper respiratory tract infections, has been implicated in asthma exacerbations and possibly asthma deaths. Although the method of transmission of rhinoviruses is disputed, several studies have demonstrated that aerosol transmission is a likely method of transmission among adults. As a first step in studies of possible airborne rhinovirus transmission, we developed methods to detect aerosolized rhinovirus by extending existing technology for detecting infectious agents in nasal specimens. METHODS: We aerosolized rhinovirus in a small aerosol chamber. Experiments were conducted with decreasing concentrations of rhinovirus. To determine the effect of UV irradiation on detection of rhinoviral aerosols, we also conducted experiments in which we exposed aerosols to a UV dose of 684 mJ/m(2). Aerosols were collected on Teflon filters and rhinovirus recovered in Qiagen AVL buffer using the Qiagen QIAamp Viral RNA Kit (Qiagen Corp., Valencia, California) followed by semi-nested RT-PCR and detection by gel electrophoresis. RESULTS: We obtained positive results from filter samples that had collected at least 1.3 TCID(50 )of aerosolized rhinovirus. Ultraviolet irradiation of airborne virus at doses much greater than those used in upper-room UV germicidal irradiation applications did not inhibit subsequent detection with the RT-PCR assay. CONCLUSION: The air sampling and extraction methodology developed in this study should be applicable to the detection of rhinovirus and other airborne viruses in the indoor air of offices and schools. This method, however, cannot distinguish UV inactivated virus from infectious viral particles.",
         "2003-01-13",
         "Myatt, Theodore A; Johnston, Sebastian L; Rudnick, Stephen; Milton, Donald K",
         "BMC Public Health",
         null,
         null,
         null,
         "document_parses/pdf_json/1e1286db212100993d03cc22374b624f7caee956.json",
         "document_parses/pmc_json/PMC140314.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC140314/",
         null
        ],
        [
         "13",
         "gi6uaa83",
         "8ae137c8da1607b3a8e4c946c07ca8bda67f88ac",
         "PMC",
         "Discovering human history from stomach bacteria",
         "10.1186/gb-2003-4-5-213",
         "PMC156578",
         "12734001.0",
         "no-cc",
         "Recent analyses of human pathogens have revealed that their evolutionary histories are congruent with the hypothesized pattern of ancient and modern human population migrations. Phylogenetic trees of strains of the bacterium Helicobacter pylori and the polyoma JC virus taken from geographically diverse groups of human beings correlate closely with relationships of the populations in which they are found.",
         "2003-04-28",
         "Disotell, Todd R",
         "Genome Biol",
         null,
         null,
         null,
         "document_parses/pdf_json/8ae137c8da1607b3a8e4c946c07ca8bda67f88ac.json",
         "document_parses/pmc_json/PMC156578.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC156578/",
         null
        ],
        [
         "14",
         "0m32ecnu",
         "23bc55d6f63fab18b02004483888db2b6a0bfa48",
         "PMC",
         "Prokaryotic-style frameshifting in a plant translation system: conservation of an unusual single-tRNA slippage event",
         "10.1093/emboj/cdg365",
         "PMC169038",
         "12881428.0",
         "green-oa",
         "Ribosomal frameshifting signals are found in mobile genetic elements, viruses and cellular genes of prokaryotes and eukaryotes. Typically they comprise a slippery sequence, X XXY YYZ, where the frameshift occurs, and a stimulatory mRNA element. Here we studied the influence of host translational environment and the identity of slippery sequence-decoding tRNAs on the frameshift mechanism. By expressing candidate signals in Escherichia coli, and in wheatgerm extracts depleted of endogenous tRNAs and supplemented with prokaryotic or eukaryotic tRNA populations, we show that when decoding AAG in the ribosomal A-site, E.coli tRNA(Lys) promotes a highly unusual single-tRNA slippage event in both prokaryotic and eukaryotic ribosomes. This event does not appear to require slippage of the adjacent P-site tRNA, although its identity is influential. Conversely, asparaginyl-tRNA promoted a dual slippage event in either system. Thus, the tRNAs themselves are the main determinants in the selection of single- or dual-tRNA slippage mechanisms. We also show for the first time that prokaryotic tRNA(Asn) is not inherently ‘unslippery’ and induces efficient frameshifting when in the context of a eukaryotic translation system.",
         "2003-08-01",
         "Napthine, Sawsan; Vidakovic, Marijana; Girnary, Roseanne; Namy, Olivier; Brierley, Ian",
         "The EMBO Journal",
         null,
         null,
         null,
         "document_parses/pdf_json/23bc55d6f63fab18b02004483888db2b6a0bfa48.json",
         "document_parses/pmc_json/PMC169038.xml.json",
         "http://europepmc.org/articles/pmc169038?pdf=render",
         null
        ],
        [
         "15",
         "le0ogx1s",
         null,
         "PMC",
         "A new recruit for the army of the men of death",
         "10.1186/gb-2003-4-7-113",
         "PMC193621",
         "12844350.0",
         "no-cc",
         "The army of the men of death, in John Bunyan's memorable phrase, has a new recruit, and fear has a new face: a face wearing a surgical mask.",
         "2003-06-27",
         "Petsko, Gregory A",
         "Genome Biol",
         null,
         null,
         null,
         null,
         null,
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC193621/",
         null
        ],
        [
         "16",
         "8qnrcgnk",
         "faaf1022ccfe93b032c5608097a53543ba24aedb",
         "PMC",
         "Heme oxygenase-1 and carbon monoxide in pulmonary medicine",
         "10.1186/1465-9921-4-7",
         "PMC193681",
         "12964953.0",
         "no-cc",
         "Heme oxygenase-1 (HO-1), an inducible stress protein, confers cytoprotection against oxidative stress in vitro and in vivo. In addition to its physiological role in heme degradation, HO-1 may influence a number of cellular processes, including growth, inflammation, and apoptosis. By virtue of anti-inflammatory effects, HO-1 limits tissue damage in response to proinflammatory stimuli and prevents allograft rejection after transplantation. The transcriptional upregulation of HO-1 responds to many agents, such as hypoxia, bacterial lipopolysaccharide, and reactive oxygen/nitrogen species. HO-1 and its constitutively expressed isozyme, heme oxygenase-2, catalyze the rate-limiting step in the conversion of heme to its metabolites, bilirubin IXα, ferrous iron, and carbon monoxide (CO). The mechanisms by which HO-1 provides protection most likely involve its enzymatic reaction products. Remarkably, administration of CO at low concentrations can substitute for HO-1 with respect to anti-inflammatory and anti-apoptotic effects, suggesting a role for CO as a key mediator of HO-1 function. Chronic, low-level, exogenous exposure to CO from cigarette smoking contributes to the importance of CO in pulmonary medicine. The implications of the HO-1/CO system in pulmonary diseases will be discussed in this review, with an emphasis on inflammatory states.",
         "2003-08-07",
         "Slebos, Dirk-Jan; Ryter, Stefan W; Choi, Augustine MK",
         "Respir Res",
         null,
         null,
         null,
         "document_parses/pdf_json/faaf1022ccfe93b032c5608097a53543ba24aedb.json",
         "document_parses/pmc_json/PMC193681.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC193681/",
         null
        ],
        [
         "17",
         "3oxzzxnd",
         "7ff45096210eeb392d51f646f5c7fe011079aaf3",
         "PMC",
         "SseG, a virulence protein that targets Salmonella to the Golgi network",
         "10.1093/emboj/cdg517",
         "PMC204495",
         "14517239.0",
         "green-oa",
         "Intracellular replication of the bacterial pathogen Salmonella enterica occurs in membrane-bound compartments called Salmonella-containing vacuoles (SCVs). Maturation of the SCV has been shown to occur by selective interactions with the endocytic pathway. We show here that after invasion of epithelial cells and migration to a perinuclear location, the majority of SCVs become surrounded by membranes of the Golgi network. This process is dependent on the Salmonella pathogenicity island 2 type III secretion system effector SseG. In infected cells, SseG was associated with the SCV and peripheral punctate structures. Only bacterial cells closely associated with the Golgi network were able to multiply; furthermore, mutation of sseG or disruption of the Golgi network inhibited intracellular bacterial growth. When expressed in epithelial cells, SseG co-localized extensively with markers of the trans-Golgi network. We identify a Golgi-targeting domain within SseG, and other regions of the protein that are required for localization of bacteria to the Golgi network. Therefore, replication of Salmonella in epithelial cells is dependent on simultaneous and selective interactions with both endocytic and secretory pathways.",
         "2003-10-01",
         "Salcedo, Suzana P.; Holden, David W.",
         "The EMBO Journal",
         null,
         null,
         null,
         "document_parses/pdf_json/7ff45096210eeb392d51f646f5c7fe011079aaf3.json",
         "document_parses/pmc_json/PMC204495.xml.json",
         "http://europepmc.org/articles/pmc204495?pdf=render",
         null
        ],
        [
         "18",
         "fy4w7xz8",
         "0104f6ceccf92ae8567a0102f89cbb976969a774",
         "PMC",
         "Association of HLA class I with severe acute respiratory syndrome coronavirus infection",
         "10.1186/1471-2350-4-9",
         "PMC212558",
         "12969506.0",
         "no-cc",
         "BACKGROUND: The human leukocyte antigen (HLA) system is widely used as a strategy in the search for the etiology of infectious diseases and autoimmune disorders. During the Taiwan epidemic of severe acute respiratory syndrome (SARS), many health care workers were infected. In an effort to establish a screening program for high risk personal, the distribution of HLA class I and II alleles in case and control groups was examined for the presence of an association to a genetic susceptibly or resistance to SARS coronavirus infection. METHODS: HLA-class I and II allele typing by PCR-SSOP was performed on 37 cases of probable SARS, 28 fever patients excluded later as probable SARS, and 101 non-infected health care workers who were exposed or possibly exposed to SARS coronavirus. An additional control set of 190 normal healthy unrelated Taiwanese was also used in the analysis. RESULTS: Woolf and Haldane Odds ratio (OR) and corrected P-value (Pc) obtained from two tails Fisher exact test were used to show susceptibility of HLA class I or class II alleles with coronavirus infection. At first, when analyzing infected SARS patients and high risk health care workers groups, HLA-B*4601 (OR = 2.08, P = 0.04, Pc = n.s.) and HLA-B*5401 (OR = 5.44, P = 0.02, Pc = n.s.) appeared as the most probable elements that may be favoring SARS coronavirus infection. After selecting only a \"severe cases\" patient group from the infected \"probable SARS\" patient group and comparing them with the high risk health care workers group, the severity of SARS was shown to be significantly associated with HLA-B*4601 (P = 0.0008 or Pc = 0.0279). CONCLUSIONS: Densely populated regions with genetically related southern Asian populations appear to be more affected by the spreading of SARS infection. Up until recently, no probable SARS patients were reported among Taiwan indigenous peoples who are genetically distinct from the Taiwanese general population, have no HLA-B* 4601 and have high frequency of HLA-B* 1301. While increase of HLA-B* 4601 allele frequency was observed in the \"Probable SARS infected\" patient group, a further significant increase of the allele was seen in the \"Severe cases\" patient group. These results appeared to indicate association of HLA-B* 4601 with the severity of SARS infection in Asian populations. Independent studies are needed to test these results.",
         "2003-09-12",
         "Lin, Marie; Tseng, Hsiang-Kuang; Trejaut, Jean A; Lee, Hui-Lin; Loo, Jun-Hun; Chu, Chen-Chung; Chen, Pei-Jan; Su, Ying-Wen; Lim, Ken Hong; Tsai, Zen-Uong; Lin, Ruey-Yi; Lin, Ruey-Shiung; Huang, Chun-Hsiung",
         "BMC Med Genet",
         null,
         null,
         null,
         "document_parses/pdf_json/0104f6ceccf92ae8567a0102f89cbb976969a774.json",
         "document_parses/pmc_json/PMC212558.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC212558/",
         null
        ],
        [
         "19",
         "jg13scgo",
         "5b44feca5d6ffaaeb66501fa84cc6dd44d06660a",
         "PMC",
         "Technical Description of RODS: A Real-time Public Health Surveillance System",
         "10.1197/jamia.m1345",
         "PMC212776",
         "12807803.0",
         "bronze-oa",
         "This report describes the design and implementation of the Real-time Outbreak and Disease Surveillance (RODS) system, a computer-based public health surveillance system for early detection of disease outbreaks. Hospitals send RODS data from clinical encounters over virtual private networks and leased lines using the Health Level 7 (HL7) message protocol. The data are sent in real time. RODS automatically classifies the registration chief complaint from the visit into one of seven syndrome categories using Bayesian classifiers. It stores the data in a relational database, aggregates the data for analysis using data warehousing techniques, applies univariate and multivariate statistical detection algorithms to the data, and alerts users of when the algorithms identify anomalous patterns in the syndrome counts. RODS also has a Web-based user interface that supports temporal and spatial analyses. RODS processes sales of over-the-counter health care products in a similar manner but receives such data in batch mode on a daily basis. RODS was used during the 2002 Winter Olympics and currently operates in two states—Pennsylvania and Utah. It has been and continues to be a resource for implementing, evaluating, and applying new methods of public health surveillance.",
         "2003-09-01",
         "Tsui, Fu-Chiang; Espino, Jeremy U.; Dato, Virginia M.; Gesteland, Per H.; Hutman, Judith; Wagner, Michael M.",
         "Journal of the American Medical Informatics Association",
         null,
         null,
         null,
         "document_parses/pdf_json/5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json",
         "document_parses/pmc_json/PMC212776.xml.json",
         "https://academic.oup.com/jamia/article-pdf/10/5/399/2352016/10-5-399.pdf",
         null
        ],
        [
         "20",
         "0qaoam29",
         "5b68a553a7cbbea13472721cd1ad617d42b40c26",
         "PMC",
         "A double epidemic model for the SARS propagation",
         "10.1186/1471-2334-3-19",
         "PMC222908",
         "12964944.0",
         "no-cc",
         "BACKGROUND: An epidemic of a Severe Acute Respiratory Syndrome (SARS) caused by a new coronavirus has spread from the Guangdong province to the rest of China and to the world, with a puzzling contagion behavior. It is important both for predicting the future of the present outbreak and for implementing effective prophylactic measures, to identify the causes of this behavior. RESULTS: In this report, we show first that the standard Susceptible-Infected-Removed (SIR) model cannot account for the patterns observed in various regions where the disease spread. We develop a model involving two superimposed epidemics to study the recent spread of the SARS in Hong Kong and in the region. We explore the situation where these epidemics may be caused either by a virus and one or several mutants that changed its tropism, or by two unrelated viruses. This has important consequences for the future: the innocuous epidemic might still be there and generate, from time to time, variants that would have properties similar to those of SARS. CONCLUSION: We find that, in order to reconcile the existing data and the spread of the disease, it is convenient to suggest that a first milder outbreak protected against the SARS. Regions that had not seen the first epidemic, or that were affected simultaneously with the SARS suffered much more, with a very high percentage of persons affected. We also find regions where the data appear to be inconsistent, suggesting that they are incomplete or do not reflect an appropriate identification of SARS patients. Finally, we could, within the framework of the model, fix limits to the future development of the epidemic, allowing us to identify landmarks that may be useful to set up a monitoring system to follow the evolution of the epidemic. The model also suggests that there might exist a SARS precursor in a large reservoir, prompting for implementation of precautionary measures when the weather cools down.",
         "2003-09-10",
         "Ng, Tuen Wai; Turinici, Gabriel; Danchin, Antoine",
         "BMC Infect Dis",
         null,
         null,
         null,
         "document_parses/pdf_json/5b68a553a7cbbea13472721cd1ad617d42b40c26.json",
         "document_parses/pmc_json/PMC222908.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC222908/",
         null
        ],
        [
         "21",
         "qj4dh6rg",
         "3ed670f60a7be2e3e2a991ea8af1fdd5fa5e2b2c",
         "PMC",
         "Cloaked similarity between HIV-1 and SARS-CoV suggests an anti-SARS strategy",
         "10.1186/1471-2180-3-20",
         "PMC222911",
         "14499001.0",
         "no-cc",
         "BACKGROUND: Severe acute respiratory syndrome (SARS) is a febrile respiratory illness. The disease has been etiologically linked to a novel coronavirus that has been named the SARS-associated coronavirus (SARS-CoV), whose genome was recently sequenced. Since it is a member of the Coronaviridae, its spike protein (S2) is believed to play a central role in viral entry by facilitating fusion between the viral and host cell membranes. The protein responsible for viral-induced membrane fusion of HIV-1 (gp41) differs in length, and has no sequence homology with S2. RESULTS: Sequence analysis reveals that the two viral proteins share the sequence motifs that construct their active conformation. These include (1) an N-terminal leucine/isoleucine zipper-like sequence, and (2) a C-terminal heptad repeat located upstream of (3) an aromatic residue-rich region juxtaposed to the (4) transmembrane segment. CONCLUSIONS: This study points to a similar mode of action for the two viral proteins, suggesting that anti-viral strategy that targets the viral-induced membrane fusion step can be adopted from HIV-1 to SARS-CoV. Recently the FDA approved Enfuvirtide, a synthetic peptide corresponding to the C-terminal heptad repeat of HIV-1 gp41, as an anti-AIDS agent. Enfuvirtide and C34, another anti HIV-1 peptide, exert their inhibitory activity by binding to a leucine/isoleucine zipper-like sequence in gp41, thus inhibiting a conformational change of gp41 required for its activation. We suggest that peptides corresponding to the C-terminal heptad repeat of the S2 protein may serve as inhibitors for SARS-CoV entry.",
         "2003-09-21",
         "Kliger, Yossef; Levanon, Erez Y",
         "BMC Microbiol",
         null,
         null,
         null,
         "document_parses/pdf_json/3ed670f60a7be2e3e2a991ea8af1fdd5fa5e2b2c.json",
         "document_parses/pmc_json/PMC222911.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC222911/",
         null
        ],
        [
         "22",
         "1wswi7us",
         "59ffca26c9d1f881e4e37e7c5af626d72d858dc4",
         "PMC",
         "Relationship of SARS-CoV to other pathogenic RNA viruses explored by tetranucleotide usage profiling",
         "10.1186/1471-2105-4-43",
         "PMC222961",
         "14499005.0",
         "no-cc",
         "BACKGROUND: The exact origin of the cause of the Severe Acute Respiratory Syndrome (SARS) is still an open question. The genomic sequence relationship of SARS-CoV with 30 different single-stranded RNA (ssRNA) viruses of various families was studied using two non-standard approaches. Both approaches began with the vectorial profiling of the tetra-nucleotide usage pattern V for each virus. In approach one, a distance measure of a vector V, based on correlation coefficient was devised to construct a relationship tree by the neighbor-joining algorithm. In approach two, a multivariate factor analysis was performed to derive the embedded tetra-nucleotide usage patterns. These patterns were subsequently used to classify the selected viruses. RESULTS: Both approaches yielded relationship outcomes that are consistent with the known virus classification. They also indicated that the genome of RNA viruses from the same family conform to a specific pattern of word usage. Based on the correlation of the overall tetra-nucleotide usage patterns, the Transmissible Gastroenteritis Virus (TGV) and the Feline CoronaVirus (FCoV) are closest to SARS-CoV. Surprisingly also, the RNA viruses that do not go through a DNA stage displayed a remarkable discrimination against the CpG and UpA di-nucleotide (z = -77.31, -52.48 respectively) and selection for UpG and CpA (z = 65.79,49.99 respectively). Potential factors influencing these biases are discussed. CONCLUSION: The study of genomic word usage is a powerful method to classify RNA viruses. The congruence of the relationship outcomes with the known classification indicates that there exist phylogenetic signals in the tetra-nucleotide usage patterns, that is most prominent in the replicase open reading frames.",
         "2003-09-20",
         "Yap, Yee Leng; Zhang, Xue Wu; Danchin, Antoine",
         "BMC Bioinformatics",
         null,
         null,
         null,
         "document_parses/pdf_json/59ffca26c9d1f881e4e37e7c5af626d72d858dc4.json",
         "document_parses/pmc_json/PMC222961.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC222961/",
         null
        ],
        [
         "23",
         "yy96yeu9",
         "c63c4d58d170136b8d3b5a66424b5ac3f73a92d9",
         "PMC",
         "Viral Discovery and Sequence Recovery Using DNA Microarrays",
         "10.1371/journal.pbio.0000002",
         "PMC261870",
         "14624234.0",
         "cc-by",
         "Because of the constant threat posed by emerging infectious diseases and the limitations of existing approaches used to identify new pathogens, there is a great demand for new technological methods for viral discovery. We describe herein a DNA microarray-based platform for novel virus identification and characterization. Central to this approach was a DNA microarray designed to detect a wide range of known viruses as well as novel members of existing viral families; this microarray contained the most highly conserved 70mer sequences from every fully sequenced reference viral genome in GenBank. During an outbreak of severe acute respiratory syndrome (SARS) in March 2003, hybridization to this microarray revealed the presence of a previously uncharacterized coronavirus in a viral isolate cultivated from a SARS patient. To further characterize this new virus, approximately 1 kb of the unknown virus genome was cloned by physically recovering viral sequences hybridized to individual array elements. Sequencing of these fragments confirmed that the virus was indeed a new member of the coronavirus family. This combination of array hybridization followed by direct viral sequence recovery should prove to be a general strategy for the rapid identification and characterization of novel viruses and emerging infectious disease.",
         "2003-11-17",
         "Wang, David; Urisman, Anatoly; Liu, Yu-Tsueng; Springer, Michael; Ksiazek, Thomas G; Erdman, Dean D; Mardis, Elaine R; Hickenbotham, Matthew; Magrini, Vincent; Eldred, James; Latreille, J. Phillipe; Wilson, Richard K; Ganem, Don; DeRisi, Joseph L",
         "PLoS Biol",
         null,
         null,
         null,
         "document_parses/pdf_json/c63c4d58d170136b8d3b5a66424b5ac3f73a92d9.json",
         "document_parses/pmc_json/PMC261870.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC261870/",
         null
        ],
        [
         "24",
         "5o38ihe0",
         "dab2d95da5307c840eb4a1e00f0186022541d63d",
         "PMC",
         "A model of tripeptidyl-peptidase I (CLN2), a ubiquitous and highly conserved member of the sedolisin family of serine-carboxyl peptidases",
         "10.1186/1472-6807-3-8",
         "PMC280685",
         "14609438.0",
         "no-cc",
         "BACKGROUND: Tripeptidyl-peptidase I, also known as CLN2, is a member of the family of sedolisins (serine-carboxyl peptidases). In humans, defects in expression of this enzyme lead to a fatal neurodegenerative disease, classical late-infantile neuronal ceroid lipofuscinosis. Similar enzymes have been found in the genomic sequences of several species, but neither systematic analyses of their distribution nor modeling of their structures have been previously attempted. RESULTS: We have analyzed the presence of orthologs of human CLN2 in the genomic sequences of a number of eukaryotic species. Enzymes with sequences sharing over 80% identity have been found in the genomes of macaque, mouse, rat, dog, and cow. Closely related, although clearly distinct, enzymes are present in fish (fugu and zebra), as well as in frogs (Xenopus tropicalis). A three-dimensional model of human CLN2 was built based mainly on the homology with Pseudomonas sp. 101 sedolisin. CONCLUSION: CLN2 is very highly conserved and widely distributed among higher organisms and may play an important role in their life cycles. The model presented here indicates a very open and accessible active site that is almost completely conserved among all known CLN2 enzymes. This result is somehow surprising for a tripeptidase where the presence of a more constrained binding pocket was anticipated. This structural model should be useful in the search for the physiological substrates of these enzymes and in the design of more specific inhibitors of CLN2.",
         "2003-11-11",
         "Wlodawer, Alexander; Durell, Stewart R; Li, Mi; Oyama, Hiroshi; Oda, Kohei; Dunn, Ben M",
         "BMC Struct Biol",
         null,
         null,
         null,
         "document_parses/pdf_json/dab2d95da5307c840eb4a1e00f0186022541d63d.json",
         "document_parses/pmc_json/PMC280685.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC280685/",
         null
        ],
        [
         "25",
         "1ul8owic",
         "75ebad1e8641e5fe2a51214f4d97742ab59a8dae",
         "PMC",
         "Air pollution and case fatality of SARS in the People's Republic of China: an ecologic study",
         "10.1186/1476-069x-2-15",
         "PMC293432",
         "14629774.0",
         "no-cc",
         "BACKGROUND: Severe acute respiratory syndrome (SARS) has claimed 349 lives with 5,327 probable cases reported in mainland China since November 2002. SARS case fatality has varied across geographical areas, which might be partially explained by air pollution level. METHODS: Publicly accessible data on SARS morbidity and mortality were utilized in the data analysis. Air pollution was evaluated by air pollution index (API) derived from the concentrations of particulate matter, sulfur dioxide, nitrogen dioxide, carbon monoxide and ground-level ozone. Ecologic analysis was conducted to explore the association and correlation between air pollution and SARS case fatality via model fitting. Partially ecologic studies were performed to assess the effects of long-term and short-term exposures on the risk of dying from SARS. RESULTS: Ecologic analysis conducted among 5 regions with 100 or more SARS cases showed that case fatality rate increased with the increment of API (case fatality = - 0.063 + 0.001 * API). Partially ecologic study based on short-term exposure demonstrated that SARS patients from regions with moderate APIs had an 84% increased risk of dying from SARS compared to those from regions with low APIs (RR = 1.84, 95% CI: 1.41–2.40). Similarly, SARS patients from regions with high APIs were twice as likely to die from SARS compared to those from regions with low APIs. (RR = 2.18, 95% CI: 1.31–3.65). Partially ecologic analysis based on long-term exposure to ambient air pollution showed the similar association. CONCLUSION: Our studies demonstrated a positive association between air pollution and SARS case fatality in Chinese population by utilizing publicly accessible data on SARS statistics and air pollution indices. Although ecologic fallacy and uncontrolled confounding effect might have biased the results, the possibility of a detrimental effect of air pollution on the prognosis of SARS patients deserves further investigation.",
         "2003-11-20",
         "Cui, Yan; Zhang, Zuo-Feng; Froines, John; Zhao, Jinkou; Wang, Hua; Yu, Shun-Zhang; Detels, Roger",
         "Environ Health",
         null,
         null,
         null,
         "document_parses/pdf_json/75ebad1e8641e5fe2a51214f4d97742ab59a8dae.json",
         "document_parses/pmc_json/PMC293432.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC293432/",
         null
        ],
        [
         "26",
         "5s6acr7m",
         "6c0dee643345c93436cfa8d58270aa887ad92e3d",
         "PMC",
         "The Virus That Changed My World",
         "10.1371/journal.pbio.0000066",
         "PMC300679",
         "14691538.0",
         "no-cc",
         "Personal account of a young virologist working in Singapore at the height of the 2003 SARS pandemic",
         "2003-12-22",
         "Poh Ng, Lisa Fong",
         "PLoS Biol",
         null,
         null,
         null,
         "document_parses/pdf_json/6c0dee643345c93436cfa8d58270aa887ad92e3d.json",
         "document_parses/pmc_json/PMC300679.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC300679/",
         null
        ],
        [
         "27",
         "5tkvsudh",
         "9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9",
         "PMC",
         "Conservation of polyamine regulation by translational frameshifting from yeast to mammals",
         "10.1093/emboj/19.8.1907",
         "PMC302018",
         "10775274.0",
         "no-cc",
         "Regulation of ornithine decarboxylase in vertebrates involves a negative feedback mechanism requiring the protein antizyme. Here we show that a similar mechanism exists in the fission yeast Schizosaccharomyces pombe. The expression of mammalian antizyme genes requires a specific +1 translational frameshift. The efficiency of the frameshift event reflects cellular polyamine levels creating the autoregulatory feedback loop. As shown here, the yeast antizyme gene and several newly identified antizyme genes from different nematodes also require a ribosomal frameshift event for their expression. Twelve nucleotides around the frameshift site are identical between S.pombe and the mammalian counterparts. The core element for this frameshifting is likely to have been present in the last common ancestor of yeast, nematodes and mammals.",
         "2000-04-17",
         "Ivanov, Ivaylo P.; Matsufuji, Senya; Murakami, Yasuko; Gesteland, Raymond F.; Atkins, John F.",
         "EMBO J",
         null,
         null,
         null,
         "document_parses/pdf_json/9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json",
         "document_parses/pmc_json/PMC302018.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC302018/",
         null
        ],
        [
         "28",
         "6lvn10f4",
         "14e0cac6e86d62859e6c9f1351ab67466d89e5b3",
         "PMC",
         "Heterogeneous nuclear ribonucleoprotein A1 regulates RNA synthesis of a cytoplasmic virus",
         "10.1093/emboj/19.17.4701",
         "PMC302072",
         "10970862.0",
         "green-oa",
         "Heterogeneous nuclear ribonucleoprotein (hnRNP A1) is involved in pre-mRNA splicing in the nucleus and translational regulation in the cytoplasm. In the present study, we demonstrate that hnRNP A1 also participates in the transcription and replication of a cytoplasmic RNA virus, mouse hepatitis virus (MHV). Overexpression of hnRNP A1 accelerated the kinetics of viral RNA synthesis, whereas the expression in the cytoplasm of a dominant-negative hnRNP A1 mutant that lacks the nuclear transport domain significantly delayed it. The hnRNP A1 mutant caused a global inhibition of viral mRNA transcription and genomic replication, and also a preferential inhibition of the replication of defective-interfering RNAs. Similar to the wild-type hnRNP A1, the hnRNP A1 mutant complexed with an MHV polymerase gene product, the nucleocapsid protein and the viral RNA. However, in contrast to the wild-type hnRNP A1, the mutant protein failed to bind a 250 kDa cellular protein, suggesting that the recruitment of cellular proteins by hnRNP A1 is important for MHV RNA synthesis. Our findings establish the importance of cellular factors in viral RNA-dependent RNA synthesis.",
         "2000-09-01",
         "Shi, Stephanie T.; Huang, Peiyong; Li, Hsin-Pai; Lai, Michael M.C.",
         "The EMBO Journal",
         null,
         null,
         null,
         "document_parses/pdf_json/14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json",
         "document_parses/pmc_json/PMC302072.xml.json",
         "http://europepmc.org/articles/pmc302072?pdf=render",
         null
        ],
        [
         "29",
         "tvxpckxo",
         "d09b79026117ec9faebba46a8d13aa9b23ec751e",
         "PMC",
         "A Method to Identify p62's UBA Domain Interacting Proteins",
         "10.1251/bpo66",
         "PMC302190",
         "14702098.0",
         "no-cc",
         "The UBA domain is a conserved sequence motif among polyubiquitin binding proteins. For the first time, we demonstrate a systematic, high throughput approach to identification of UBA domain-interacting proteins from a proteome-wide perspective. Using the rabbit reticulocyte lysate in vitro expression cloning system, we have successfully identified eleven proteins that interact with p62’s UBA domain, and the majority of the eleven proteins are associated with neurodegenerative disorders, such as Alzheimer’s disease. Therefore, p62 may play a novel regulatory role through its UBA domain. Our approach provides an easy route to the characterization of UBA domain interacting proteins and its application will unfold the important roles that the UBA domain plays.",
         "2003-12-12",
         "Pridgeon, Julia W.; Geetha, Thangiah; Wooten, Marie W.",
         "Biol Proced Online",
         null,
         null,
         null,
         "document_parses/pdf_json/d09b79026117ec9faebba46a8d13aa9b23ec751e.json",
         "document_parses/pmc_json/PMC302190.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC302190/",
         null
        ],
        [
         "30",
         "mcuixluu",
         "44102e3e69e70ad2a73e753133283334ce1f8736",
         "PMC",
         "Vaccinia virus infection disrupts microtubule organization and centrosome function",
         "10.1093/emboj/19.15.3932",
         "PMC306617",
         "10921875.0",
         "green-oa",
         "We examined the role of the microtubule cytoskeleton during vaccinia virus infection. We found that newly assembled virus particles accumulate in the vicinity of the microtubule-organizing centre in a microtubule- and dynein–dynactin complex-dependent fashion. Microtubules are required for efficient intracellular mature virus (IMV) formation and are essential for intracellular enveloped virus (IEV) assembly. As infection proceeds, the microtubule cytoskeleton becomes dramatically reorganized in a fashion reminiscent of overexpression of microtubule-associated proteins (MAPs). Consistent with this, we report that the vaccinia proteins A10L and L4R have MAP-like properties and mediate direct binding of viral cores to microtubules in vitro. In addition, vaccinia infection also results in severe reduction of proteins at the centrosome and loss of centrosomal microtubule nucleation efficiency. This represents the first example of viral-induced disruption of centrosome function. Further studies with vaccinia will provide insights into the role of microtubules during viral pathogenesis and regulation of centrosome function.",
         "2000-08-01",
         "Ploubidou, Aspasia; Moreau, Violaine; Ashman, Keith; Reckmann, Inge; González, Cayetano; Way, Michael",
         "The EMBO Journal",
         null,
         null,
         null,
         "document_parses/pdf_json/44102e3e69e70ad2a73e753133283334ce1f8736.json",
         "document_parses/pmc_json/PMC306617.xml.json",
         "http://europepmc.org/articles/pmc306617?pdf=render",
         null
        ],
        [
         "31",
         "6iu1dtyl",
         "6e8517cb25ff228cb48da90a9c97d9c274dc8031",
         "PMC",
         "The site of origin of the 1918 influenza pandemic and its public health implications",
         "10.1186/1479-5876-2-3",
         "PMC340389",
         "14733617.0",
         "no-cc",
         null,
         "2004-01-20",
         "Barry, John M",
         "J Transl Med",
         null,
         null,
         null,
         "document_parses/pdf_json/6e8517cb25ff228cb48da90a9c97d9c274dc8031.json",
         null,
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC340389/",
         null
        ],
        [
         "32",
         "ri5v6u4x",
         "06559dd625491d6474ee88f08c12fc17c1830995",
         "PMC",
         "Towards evidence-based, GIS-driven national spatial health information infrastructure and surveillance services in the United Kingdom",
         "10.1186/1476-072x-3-1",
         "PMC343292",
         "14748927.0",
         "no-cc",
         "The term \"Geographic Information Systems\" (GIS) has been added to MeSH in 2003, a step reflecting the importance and growing use of GIS in health and healthcare research and practices. GIS have much more to offer than the obvious digital cartography (map) functions. From a community health perspective, GIS could potentially act as powerful evidence-based practice tools for early problem detection and solving. When properly used, GIS can: inform and educate (professionals and the public); empower decision-making at all levels; help in planning and tweaking clinically and cost-effective actions, in predicting outcomes before making any financial commitments and ascribing priorities in a climate of finite resources; change practices; and continually monitor and analyse changes, as well as sentinel events. Yet despite all these potentials for GIS, they remain under-utilised in the UK National Health Service (NHS). This paper has the following objectives: (1) to illustrate with practical, real-world scenarios and examples from the literature the different GIS methods and uses to improve community health and healthcare practices, e.g., for improving hospital bed availability, in community health and bioterrorism surveillance services, and in the latest SARS outbreak; (2) to discuss challenges and problems currently hindering the wide-scale adoption of GIS across the NHS; and (3) to identify the most important requirements and ingredients for addressing these challenges, and realising GIS potential within the NHS, guided by related initiatives worldwide. The ultimate goal is to illuminate the road towards implementing a comprehensive national, multi-agency spatio-temporal health information infrastructure functioning proactively in real time. The concepts and principles presented in this paper can be also applied in other countries, and on regional (e.g., European Union) and global levels.",
         "2004-01-28",
         "Boulos, Maged N Kamel",
         "Int J Health Geogr",
         null,
         null,
         null,
         "document_parses/pdf_json/06559dd625491d6474ee88f08c12fc17c1830995.json",
         "document_parses/pmc_json/PMC343292.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC343292/",
         null
        ],
        [
         "33",
         "kuybfc1y",
         "d81ae79ebc6462fd3de075e220992f1aae1f06ae",
         "PMC",
         "Descriptive review of geographic mapping of severe acute respiratory syndrome (SARS) on the Internet",
         "10.1186/1476-072x-3-2",
         "PMC343293",
         "14748926.0",
         "no-cc",
         "From geographic mapping at different scales to location-based alerting services, geoinformatics plays an important role in the study and control of global outbreaks like severe acute respiratory syndrome (SARS). This paper reviews several geographic mapping efforts of SARS on the Internet that employ a variety of techniques like choropleth rendering, graduated circles, graduated pie charts, buffering, overlay analysis and animation. The aim of these mapping services is to educate the public (especially travellers to potentially at-risk areas) and assist public health authorities in analysing the spatial and temporal trends and patterns of SARS and in assessing/revising current control measures.",
         "2004-01-28",
         "Boulos, Maged N Kamel",
         "Int J Health Geogr",
         null,
         null,
         null,
         "document_parses/pdf_json/d81ae79ebc6462fd3de075e220992f1aae1f06ae.json",
         "document_parses/pmc_json/PMC343293.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC343293/",
         null
        ],
        [
         "34",
         "ng4rrdte",
         "87390d2ae28407b3e03e60a6b24a7fd99ed7229a",
         "PMC",
         "Pro/con clinical debate: Steroids are a key component in the treatment of SARS",
         "10.1186/cc2452",
         "PMC420028",
         "15025770.0",
         "no-cc",
         "SARS (severe acute respiratory syndrome) proved an enormous physical and emotional challenge to frontline health care workers throughout the world in late 2002 through to mid 2003. A large percentage of patients (many being health care workers themselves) became critically ill. Unfortunately, clinicians caring for these individuals did not have the advantage of previous experience or research data on which to base treatment decisions. As a result, at least early in the outbreak, a 'best guess approach' and/or anecdotes drove therapy. In many centres systemic steroids, which carry many potential downsides, became a mainstay of therapy. In this issue of Critical Care, two groups that have frontline experience of SARS debate the role of steroids. Let us hope and pray together that we never have the patient population needed to resolve the questions the two sides raise.",
         "2004-01-26",
         "Gomersall, Charles D; Kargel, Marcus J; Lapinsky, Stephen E",
         "Crit Care",
         null,
         null,
         null,
         "document_parses/pdf_json/87390d2ae28407b3e03e60a6b24a7fd99ed7229a.json",
         "document_parses/pmc_json/PMC420028.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC420028/",
         null
        ],
        [
         "35",
         "1769ovyk",
         "c5131e5f5c6000ec84139edc64778a6f1d391b83",
         "PMC",
         "8th Annual Toronto Critical Care Medicine Symposium, 30 October–1 November 2003, Toronto, Ontario, Canada",
         "10.1186/cc2429",
         "PMC420071",
         "14975048.0",
         "no-cc",
         null,
         "2004-01-02",
         "Granton, Jeff; Granton, John",
         "Crit Care",
         null,
         null,
         null,
         "document_parses/pdf_json/c5131e5f5c6000ec84139edc64778a6f1d391b83.json",
         "document_parses/pmc_json/PMC420071.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC420071/",
         null
        ],
        [
         "36",
         "tixxm78q",
         "8a7d5de5ea680e784ab2bd877240bf09e4c1c02d",
         "PMC",
         "Recently published papers: all the usual suspects and carbon dioxide",
         "10.1186/cc2449",
         "PMC420074",
         "14975037.0",
         "no-cc",
         null,
         "2004-01-02",
         "Ball, Jonathan",
         "Crit Care",
         null,
         null,
         null,
         "document_parses/pdf_json/8a7d5de5ea680e784ab2bd877240bf09e4c1c02d.json",
         "document_parses/pmc_json/PMC420074.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC420074/",
         null
        ],
        [
         "37",
         "t35n7bk9",
         "30a4842a2e257f725cc041e32fd682c495111a50",
         "PMC",
         "Multi-faceted, multi-versatile microarray: simultaneous detection of many viruses and their expression profiles",
         "10.1186/1742-4690-1-11",
         "PMC420498",
         "15169556.0",
         "no-cc",
         "There are hundreds of viruses that infect different human organs and cause diseases. Some fatal emerging viral infections have become serious public health issues worldwide. Early diagnosis and subsequent treatment are therefore essential for fighting viral infections. Current diagnostic techniques frequently employ polymerase chain reaction (PCR)-based methods to quickly detect the pathogenic viruses and establish the etiology of the disease or illness. However, the fast PCR method suffers from many drawbacks such as a high false-positive rate and the ability to detect only one or a few gene targets at a time. Microarray technology solves the problems of the PCR limitations and can be effectively applied to all fields of molecular medicine. Recently, a report in Retrovirology described a multi-virus DNA array that contains more than 250 open reading frames from eight human viruses including human immunodeficiency virus type 1. This array can be used to detect multiple viral co-infections in cells and in vivo. Another benefit of this kind of multi-virus array is in studying promoter activity and viral gene expression and correlating such readouts with the progression of disease and reactivation of latent infections. Thus, the virus DNA-chip development reported in Retrovirology is an important advance in diagnostic application which could be a potent clinical tool for characterizing viral co-infections in AIDS as well as other patients.",
         "2004-05-26",
         "Shieh, Biehuoy; Li, Ching",
         "Retrovirology",
         null,
         null,
         null,
         "document_parses/pdf_json/30a4842a2e257f725cc041e32fd682c495111a50.json",
         "document_parses/pmc_json/PMC420498.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC420498/",
         null
        ],
        [
         "38",
         "pwsvhitd",
         "e4b48ce0579f908da6dd3289bb2cc262dbdeae72",
         "PMC",
         "Detection of hepatitis C virus in the nasal secretions of an intranasal drug-user",
         "10.1186/1476-0711-3-6",
         "PMC421742",
         "15132748.0",
         "no-cc",
         "BACKGROUND: One controversial source of infection for hepatitis C virus (HCV) involves the sharing of contaminated implements, such as straws or spoons, used to nasally inhale cocaine and other powdered drugs. An essential precondition for this mode of transmission is the presence of HCV in the nasal secretions of intranasal drug users. METHODS: Blood and nasal secretion samples were collected from five plasma-positive chronic intranasal drug users and tested for HCV RNA using RT-PCR. RESULTS: HCV was detected in all five blood samples and in the nasal secretions of the subject with the highest serum viral load. CONCLUSIONS: This study is the first to demonstrate the presence of HCV in nasal secretions. This finding has implications for potential transmission of HCV through contact with contaminated nasal secretions.",
         "2004-05-07",
         "McMahon, James M; Simm, Malgorzata; Milano, Danielle; Clatts, Michael",
         "Ann Clin Microbiol Antimicrob",
         null,
         null,
         null,
         "document_parses/pdf_json/e4b48ce0579f908da6dd3289bb2cc262dbdeae72.json",
         "document_parses/pmc_json/PMC421742.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC421742/",
         null
        ],
        [
         "39",
         "wt8zfqk0",
         "52ce19fe79de46d9bb0fa7e75a0631fed2d3efd2",
         "PMC",
         "Bioinformatics analysis of SARS coronavirus genome polymorphism",
         "10.1186/1471-2105-5-65",
         "PMC434493",
         "15161495.0",
         "no-cc",
         "BACKGROUND: We have compared 38 isolates of the SARS-CoV complete genome. The main goal was twofold: first, to analyze and compare nucleotide sequences and to identify positions of single nucleotide polymorphism (SNP), insertions and deletions, and second, to group them according to sequence similarity, eventually pointing to phylogeny of SARS-CoV isolates. The comparison is based on genome polymorphism such as insertions or deletions and the number and positions of SNPs. RESULTS: The nucleotide structure of all 38 isolates is presented. Based on insertions and deletions and dissimilarity due to SNPs, the dataset of all the isolates has been qualitatively classified into three groups each having their own subgroups. These are the A-group with \"regular\" isolates (no insertions / deletions except for 5' and 3' ends), the B-group of isolates with \"long insertions\", and the C-group of isolates with \"many individual\" insertions and deletions. The isolate with the smallest average number of SNPs, compared to other isolates, has been identified (TWH). The density distribution of SNPs, insertions and deletions for each group or subgroup, as well as cumulatively for all the isolates is also presented, along with the gene map for TWH. Since individual SNPs may have occurred at random, positions corresponding to multiple SNPs (occurring in two or more isolates) are identified and presented. This result revises some previous results of a similar type. Amino acid changes caused by multiple SNPs are also identified (for the annotated sequences, as well as presupposed amino acid changes for non-annotated ones). Exact SNP positions for the isolates in each group or subgroup are presented. Finally, a phylogenetic tree for the SARS-CoV isolates has been produced using the CLUSTALW program, showing high compatibility with former qualitative classification. CONCLUSIONS: The comparative study of SARS-CoV isolates provides essential information for genome polymorphism, indication of strain differences and variants evolution. It may help with the development of effective treatment.",
         "2004-05-25",
         "Pavlović-Lažetić, Gordana M; Mitić, Nenad S; Beljanski, Miloš V",
         "BMC Bioinformatics",
         null,
         null,
         null,
         "document_parses/pdf_json/52ce19fe79de46d9bb0fa7e75a0631fed2d3efd2.json",
         "document_parses/pmc_json/PMC434493.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC434493/",
         null
        ],
        [
         "40",
         "5gsbtfag",
         "b499a41ad1d85a0e071b59e60065c874e534911d",
         "PMC",
         "Coronavirus 3CL(pro )proteinase cleavage sites: Possible relevance to SARS virus pathology",
         "10.1186/1471-2105-5-72",
         "PMC442122",
         "15180906.0",
         "no-cc",
         "BACKGROUND: Despite the passing of more than a year since the first outbreak of Severe Acute Respiratory Syndrome (SARS), efficient counter-measures are still few and many believe that reappearance of SARS, or a similar disease caused by a coronavirus, is not unlikely. For other virus families like the picornaviruses it is known that pathology is related to proteolytic cleavage of host proteins by viral proteinases. Furthermore, several studies indicate that virus proliferation can be arrested using specific proteinase inhibitors supporting the belief that proteinases are indeed important during infection. Prompted by this, we set out to analyse and predict cleavage by the coronavirus main proteinase using computational methods. RESULTS: We retrieved sequence data on seven fully sequenced coronaviruses and identified the main 3CL proteinase cleavage sites in polyproteins using alignments. A neural network was trained to recognise the cleavage sites in the genomes obtaining a sensitivity of 87.0% and a specificity of 99.0%. Several proteins known to be cleaved by other viruses were submitted to prediction as well as proteins suspected relevant in coronavirus pathology. Cleavage sites were predicted in proteins such as the cystic fibrosis transmembrane conductance regulator (CFTR), transcription factors CREB-RP and OCT-1, and components of the ubiquitin pathway. CONCLUSIONS: Our prediction method NetCorona predicts coronavirus cleavage sites with high specificity and several potential cleavage candidates were identified which might be important to elucidate coronavirus pathology. Furthermore, the method might assist in design of proteinase inhibitors for treatment of SARS and possible future diseases caused by coronaviruses. It is made available for public use at our website: .",
         "2004-06-06",
         "Kiemer, Lars; Lund, Ole; Brunak, Søren; Blom, Nikolaj",
         "BMC Bioinformatics",
         null,
         null,
         null,
         "document_parses/pdf_json/b499a41ad1d85a0e071b59e60065c874e534911d.json",
         "document_parses/pmc_json/PMC442122.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC442122/",
         null
        ],
        [
         "41",
         "fpj5urao",
         "e9c78584c08ba79d735e150eff98297eb57f12dd; 4a228865af2c19adf9386a5b04cca3ceb3c8683d; cdb29ec7a9029d22f6fbf7ee04543819591acdc2",
         "PMC",
         "Moderate mutation rate in the SARS coronavirus genome and its implications",
         "10.1186/1471-2148-4-21",
         "PMC446188",
         "15222897.0",
         "no-cc",
         "BACKGROUND: The outbreak of severe acute respiratory syndrome (SARS) caused a severe global epidemic in 2003 which led to hundreds of deaths and many thousands of hospitalizations. The virus causing SARS was identified as a novel coronavirus (SARS-CoV) and multiple genomic sequences have been revealed since mid-April, 2003. After a quiet summer and fall in 2003, the newly emerged SARS cases in Asia, particularly the latest cases in China, are reinforcing a wide-spread belief that the SARS epidemic would strike back. With the understanding that SARS-CoV might be with humans for years to come, knowledge of the evolutionary mechanism of the SARS-CoV, including its mutation rate and emergence time, is fundamental to battle this deadly pathogen. To date, the speed at which the deadly virus evolved in nature and the elapsed time before it was transmitted to humans remains poorly understood. RESULTS: Sixteen complete genomic sequences with available clinical histories during the SARS outbreak were analyzed. After careful examination of multiple-sequence alignment, 114 single nucleotide variations were identified. To minimize the effects of sequencing errors and additional mutations during the cell culture, three strategies were applied to estimate the mutation rate by 1) using the closely related sequences as background controls; 2) adjusting the divergence time for cell culture; or 3) using the common variants only. The mutation rate in the SARS-CoV genome was estimated to be 0.80 – 2.38 × 10(-3 )nucleotide substitution per site per year which is in the same order of magnitude as other RNA viruses. The non-synonymous and synonymous substitution rates were estimated to be 1.16 – 3.30 × 10(-3 )and 1.67 – 4.67 × 10(-3 )per site per year, respectively. The most recent common ancestor of the 16 sequences was inferred to be present as early as the spring of 2002. CONCLUSIONS: The estimated mutation rates in the SARS-CoV using multiple strategies were not unusual among coronaviruses and moderate compared to those in other RNA viruses. All estimates of mutation rates led to the inference that the SARS-CoV could have been with humans in the spring of 2002 without causing a severe epidemic.",
         "2004-06-28",
         "Zhao, Zhongming; Li, Haipeng; Wu, Xiaozhuang; Zhong, Yixi; Zhang, Keqin; Zhang, Ya-Ping; Boerwinkle, Eric; Fu, Yun-Xin",
         "BMC Evol Biol",
         null,
         null,
         null,
         "document_parses/pdf_json/e9c78584c08ba79d735e150eff98297eb57f12dd.json; document_parses/pdf_json/4a228865af2c19adf9386a5b04cca3ceb3c8683d.json; document_parses/pdf_json/cdb29ec7a9029d22f6fbf7ee04543819591acdc2.json",
         "document_parses/pmc_json/PMC446188.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC446188/",
         null
        ],
        [
         "42",
         "wutnzzhg",
         "017ca5bdac589a37196df7b8e077c4c371ab32da",
         "PMC",
         "Pro/con clinical debate: Isolation precautions for all intensive care unit patients with methicillin-resistant Staphylococcus aureus colonization are essential",
         "10.1186/cc2817",
         "PMC468889",
         "15153232.0",
         "no-cc",
         "Antibiotic-resistant bacteria are an increasingly common problem in intensive care units (ICUs), and they are capable of impacting on patient outcome, the ICU's budget and bed availability. This issue, coupled with recent outbreaks of illnesses that pose a direct risk to ICU staff (such as SARS [severe acute respiratory syndrome]), has led to renewed emphasis on infection control measures and practitioners in the ICU. Infection control measures frequently cause clinicians to practice in a more time consuming way. As a result it is not surprising that ensuring compliance with these measures is not always easy, particularly when their benefit is not immediately obvious. In this issue of Critical Care, two experts face off over the need to isolate patients infected with methicillin-resistant Staphylococcus aureus.",
         "2004-02-19",
         "Farr, Barry M; Bellingan, Geoffrey",
         "Crit Care",
         null,
         null,
         null,
         "document_parses/pdf_json/017ca5bdac589a37196df7b8e077c4c371ab32da.json",
         "document_parses/pmc_json/PMC468889.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC468889/",
         null
        ],
        [
         "43",
         "eiqypt0m",
         "6a8ac55ea2a1fbd99deb683e24dd986e55e707b3",
         "PMC",
         "Herpes simplex virus type 1 and normal protein permeability in the lungs of critically ill patients: a case for low pathogenicity?",
         "10.1186/cc2850",
         "PMC468896",
         "15153242.0",
         "no-cc",
         "INTRODUCTION: The pathogenicity of late respiratory infections with herpes simplex virus type 1 (HSV-1) in the critically ill is unclear. METHODS: In four critically ill patients with persistent pulmonary infiltrates of unknown origin and isolation of HSV-1 from tracheal aspirate or bronchoalveolar lavage fluid, at 7 (1–11) days after start of mechanical ventilatory support, a pulmonary leak index (PLI) for (67)Gallium ((67)Ga)-transferrin (upper limit of normal 14.1 × 10(-3)/min) was measured. RESULTS: The PLI ranged between 7.5 and 14.0 × 10(-3)/min in the study patients. Two patients received a course of acyclovir and all survived. CONCLUSIONS: The normal capillary permeability observed in the lungs argues against pathogenicity of HSV-1 in the critically ill, and favors that isolation of the virus reflects reactivation in the course of serious illness and immunodepresssion, rather than primary or superimposed infection in the lungs.",
         "2004-03-31",
         "Verheij, Joanne; Groeneveld, AB Johan; Beishuizen, Albertus; Lingen, Arthur van; Simoons-Smit, Alberdina M; van Schijndel, Rob JM Strack",
         "Crit Care",
         null,
         null,
         null,
         "document_parses/pdf_json/6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json",
         "document_parses/pmc_json/PMC468896.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC468896/",
         null
        ],
        [
         "44",
         "lwla5ugt",
         "8970ef10049816050cb1bc5fbf600865a5996ad4",
         "PMC",
         "Base-By-Base: Single nucleotide-level analysis of whole viral genome alignments",
         "10.1186/1471-2105-5-96",
         "PMC481056",
         "15253776.0",
         "no-cc",
         "BACKGROUND: With ever increasing numbers of closely related virus genomes being sequenced, it has become desirable to be able to compare two genomes at a level more detailed than gene content because two strains of an organism may share the same set of predicted genes but still differ in their pathogenicity profiles. For example, detailed comparison of multiple isolates of the smallpox virus genome (each approximately 200 kb, with 200 genes) is not feasible without new bioinformatics tools. RESULTS: A software package, Base-By-Base, has been developed that provides visualization tools to enable researchers to 1) rapidly identify and correct alignment errors in large, multiple genome alignments; and 2) generate tabular and graphical output of differences between the genomes at the nucleotide level. Base-By-Base uses detailed annotation information about the aligned genomes and can list each predicted gene with nucleotide differences, display whether variations occur within promoter regions or coding regions and whether these changes result in amino acid substitutions. Base-By-Base can connect to our mySQL database (Virus Orthologous Clusters; VOCs) to retrieve detailed annotation information about the aligned genomes or use information from text files. CONCLUSION: Base-By-Base enables users to quickly and easily compare large viral genomes; it highlights small differences that may be responsible for important phenotypic differences such as virulence. It is available via the Internet using Java Web Start and runs on Macintosh, PC and Linux operating systems with the Java 1.4 virtual machine.",
         "2004-07-14",
         "Brodie, Ryan; Smith, Alex J; Roper, Rachel L; Tcherepanov, Vasily; Upton, Chris",
         "BMC Bioinformatics",
         null,
         null,
         null,
         "document_parses/pdf_json/8970ef10049816050cb1bc5fbf600865a5996ad4.json",
         "document_parses/pmc_json/PMC481056.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC481056/",
         null
        ],
        [
         "45",
         "8zwsi4nk",
         "e3af2ca43010f59c3d1bb731abd011e3dd0fc51c",
         "PMC",
         "Date of origin of the SARS coronavirus strains",
         "10.1186/1471-2334-4-3",
         "PMC516801",
         "15028113.0",
         "no-cc",
         "BACKGROUND: A new respiratory infectious epidemic, severe acute respiratory syndrome (SARS), broke out and spread throughout the world. By now the putative pathogen of SARS has been identified as a new coronavirus, a single positive-strand RNA virus. RNA viruses commonly have a high rate of genetic mutation. It is therefore important to know the mutation rate of the SARS coronavirus as it spreads through the population. Moreover, finding a date for the last common ancestor of SARS coronavirus strains would be useful for understanding the circumstances surrounding the emergence of the SARS pandemic and the rate at which SARS coronavirus diverge. METHODS: We propose a mathematical model to estimate the evolution rate of the SARS coronavirus genome and the time of the last common ancestor of the sequenced SARS strains. Under some common assumptions and justifiable simplifications, a few simple equations incorporating the evolution rate (K) and time of the last common ancestor of the strains (T(0)) can be deduced. We then implemented the least square method to estimate K and T(0 )from the dataset of sequences and corresponding times. Monte Carlo stimulation was employed to discuss the results. RESULTS: Based on 6 strains with accurate dates of host death, we estimated the time of the last common ancestor to be about August or September 2002, and the evolution rate to be about 0.16 base/day, that is, the SARS coronavirus would on average change a base every seven days. We validated our method by dividing the strains into two groups, which coincided with the results from comparative genomics. CONCLUSION: The applied method is simple to implement and avoid the difficulty and subjectivity of choosing the root of phylogenetic tree. Based on 6 strains with accurate date of host death, we estimated a time of the last common ancestor, which is coincident with epidemic investigations, and an evolution rate in the same range as that reported for the HIV-1 virus.",
         "2004-02-06",
         "Lu, Hongchao; Zhao, Yi; Zhang, Jingfen; Wang, Yuelan; Li, Wei; Zhu, Xiaopeng; Sun, Shiwei; Xu, Jingyi; Ling, Lunjiang; Cai, Lun; Bu, Dongbo; Chen, Runsheng",
         "BMC Infect Dis",
         null,
         null,
         null,
         "document_parses/pdf_json/e3af2ca43010f59c3d1bb731abd011e3dd0fc51c.json",
         "document_parses/pmc_json/PMC516801.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC516801/",
         null
        ],
        [
         "46",
         "t20z4mtt",
         "26bdc177821e0161d138236656dafefe2f9f6900",
         "PMC",
         "Mutations in the IGF-II pathway that confer resistance to lytic reovirus infection",
         "10.1186/1471-2121-5-32",
         "PMC517494",
         "15333144.0",
         "cc-by",
         "BACKGROUND: Viruses are obligate intracellular parasites and rely upon the host cell for different steps in their life cycles. The characterization of cellular genes required for virus infection and/or cell killing will be essential for understanding viral life cycles, and may provide cellular targets for new antiviral therapies. RESULTS: A gene entrapment approach was used to identify candidate cellular genes that affect reovirus infection or virus induced cell lysis. Four of the 111 genes disrupted in clones selected for resistance to infection by reovirus type 1 involved the insulin growth factor-2 (IGF-II) pathway, including: the mannose-6-phosphate/IGF2 receptor (Igf2r), a protease associated with insulin growth factor binding protein 5 (Prss11), and the CTCF transcriptional regulator (Ctcf). The disruption of Ctcf, which encodes a repressor of Igf2, was associated with enhanced Igf2 gene expression. Plasmids expressing either the IGF-II pro-hormone or IGF-II without the carboxy terminal extension (E)-peptide sequence independently conferred high levels of cellular resistance to reovirus infection. Forced IGF-II expression results in a block in virus disassembly. In addition, Ctcf disruption and forced Igf2 expression both enabled cells to proliferate in soft agar, a phenotype associated with malignant growth in vivo. CONCLUSION: These results indicate that IGF-II, and by inference other components of the IGF-II signalling pathway, can confer resistance to lytic reovirus infection. This report represents the first use of gene entrapment to identify host factors affecting virus infection. Concomitant transformation observed in some virus resistant cells illustrates a potential mechanism of carcinogenesis associated with chronic virus infection.",
         "2004-08-27",
         "Sheng, Jinsong; Organ, Edward L; Hao, Chuanming; Wells, K Sam; Ruley, H Earl; Rubin, Donald H",
         "BMC Cell Biol",
         null,
         null,
         null,
         "document_parses/pdf_json/26bdc177821e0161d138236656dafefe2f9f6900.json",
         "document_parses/pmc_json/PMC517494.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC517494/",
         null
        ],
        [
         "47",
         "0gmtnkbh",
         "def1cf77e1ef84f4373a342e23145be05ec5e226",
         "PMC",
         "Mutational dynamics of the SARS coronavirus in cell culture and human populations isolated in 2003",
         "10.1186/1471-2334-4-32",
         "PMC517714",
         "15347429.0",
         "cc-by",
         "BACKGROUND: The SARS coronavirus is the etiologic agent for the epidemic of the Severe Acute Respiratory Syndrome. The recent emergence of this new pathogen, the careful tracing of its transmission patterns, and the ability to propagate in culture allows the exploration of the mutational dynamics of the SARS-CoV in human populations. METHODS: We sequenced complete SARS-CoV genomes taken from primary human tissues (SIN3408, SIN3725V, SIN3765V), cultured isolates (SIN848, SIN846, SIN842, SIN845, SIN847, SIN849, SIN850, SIN852, SIN3408L), and five consecutive Vero cell passages (SIN2774_P1, SIN2774_P2, SIN2774_P3, SIN2774_P4, SIN2774_P5) arising from SIN2774 isolate. These represented individual patient samples, serial in vitro passages in cell culture, and paired human and cell culture isolates. Employing a refined mutation filtering scheme and constant mutation rate model, the mutation rates were estimated and the possible date of emergence was calculated. Phylogenetic analysis was used to uncover molecular relationships between the isolates. RESULTS: Close examination of whole genome sequence of 54 SARS-CoV isolates identified before 14(th )October 2003, including 22 from patients in Singapore, revealed the mutations engendered during human-to-Vero and Vero-to-human transmission as well as in multiple Vero cell passages in order to refine our analysis of human-to-human transmission. Though co-infection by different quasipecies in individual tissue samples is observed, the in vitro mutation rate of the SARS-CoV in Vero cell passage is negligible. The in vivo mutation rate, however, is consistent with estimates of other RNA viruses at approximately 5.7 × 10(-6 )nucleotide substitutions per site per day (0.17 mutations per genome per day), or two mutations per human passage (adjusted R-square = 0.4014). Using the immediate Hotel M contact isolates as roots, we observed that the SARS epidemic has generated four major genetic groups that are geographically associated: two Singapore isolates, one Taiwan isolate, and one North China isolate which appears most closely related to the putative SARS-CoV isolated from a palm civet. Non-synonymous mutations are centered in non-essential ORFs especially in structural and antigenic genes such as the S and M proteins, but these mutations did not distinguish the geographical groupings. However, no non-synonymous mutations were found in the 3CLpro and the polymerase genes. CONCLUSIONS: Our results show that the SARS-CoV is well adapted to growth in culture and did not appear to undergo specific selection in human populations. We further assessed that the putative origin of the SARS epidemic was in late October 2002 which is consistent with a recent estimate using cases from China. The greater sequence divergence in the structural and antigenic proteins and consistent deletions in the 3' – most portion of the viral genome suggest that certain selection pressures are interacting with the functional nature of these validated and putative ORFs.",
         "2004-09-06",
         "Vega, Vinsensius B; Ruan, Yijun; Liu, Jianjun; Lee, Wah Heng; Wei, Chia Lin; Se-Thoe, Su Yun; Tang, Kin Fai; Zhang, Tao; Kolatkar, Prasanna R; Ooi, Eng Eong; Ling, Ai Ee; Stanton, Lawrence W; Long, Philip M; Liu, Edison T",
         "BMC Infect Dis",
         null,
         null,
         null,
         "document_parses/pdf_json/def1cf77e1ef84f4373a342e23145be05ec5e226.json",
         "document_parses/pmc_json/PMC517714.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC517714/",
         null
        ],
        [
         "48",
         "rrhh2alf",
         "45d2d838cb3d5ae2dabd7bb7c82329b398d1c65f",
         "PMC",
         "A human in vitro model system for investigating genome-wide host responses to SARS coronavirus infection",
         "10.1186/1471-2334-4-34",
         "PMC518965",
         "15357874.0",
         "cc-by",
         "BACKGROUND: The molecular basis of severe acute respiratory syndrome (SARS) coronavirus (CoV) induced pathology is still largely unclear. Many SARS patients suffer respiratory distress brought on by interstitial infiltration and frequently show peripheral blood lymphopenia and occasional leucopenia. One possible cause of this could be interstitial inflammation, following a localized host response. In this study, we therefore examine the immune response of SARS-CoV in human peripheral blood mononuclear cells (PBMCs) over the first 24 hours. METHODS: PBMCs from normal healthy donors were inoculated in vitro with SARS-CoV and the viral replication kinetics was studied by real-time quantitative assays. SARS-CoV specific gene expression changes were examined by high-density oligonucleotide array analysis. RESULTS: We observed that SARS-CoV was capable of infecting and replicating in PBMCs and the kinetics of viral replication was variable among the donors. SARS-CoV antibody binding assays indicated that SARS specific antibodies inhibited SARS-CoV viral replication. Array data showed monocyte-macrophage cell activation, coagulation pathway upregulation and cytokine production together with lung trafficking chemokines such as IL8 and IL17, possibly activated through the TLR9 signaling pathway; that mimicked clinical features of the disease. CONCLUSIONS: The identification of human blood mononuclear cells as a direct target of SARS-CoV in the model system described here provides a new insight into disease pathology and a tool for investigating the host response and mechanisms of pathogenesis.",
         "2004-09-09",
         "Ng, Lisa FP; Hibberd, Martin L; Ooi, Eng-Eong; Tang, Kin-Fai; Neo, Soek-Ying; Tan, Jenny; Krishna Murthy, Karuturi R; Vega, Vinsensius B; Chia, Jer-Ming; Liu, Edison T; Ren, Ee-Chee",
         "BMC Infect Dis",
         null,
         null,
         null,
         "document_parses/pdf_json/45d2d838cb3d5ae2dabd7bb7c82329b398d1c65f.json",
         "document_parses/pmc_json/PMC518965.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC518965/",
         null
        ],
        [
         "49",
         "sgmk96vr",
         "367af6bb9a8bbda02206506a819656bb6f14ce4c",
         "PMC",
         "Logistics of community smallpox control through contact tracing and ring vaccination: a stochastic network model",
         "10.1186/1471-2458-4-34",
         "PMC520756",
         "15298713.0",
         "cc-by",
         "BACKGROUND: Previous smallpox ring vaccination models based on contact tracing over a network suggest that ring vaccination would be effective, but have not explicitly included response logistics and limited numbers of vaccinators. METHODS: We developed a continuous-time stochastic simulation of smallpox transmission, including network structure, post-exposure vaccination, vaccination of contacts of contacts, limited response capacity, heterogeneity in symptoms and infectiousness, vaccination prior to the discontinuation of routine vaccination, more rapid diagnosis due to public awareness, surveillance of asymptomatic contacts, and isolation of cases. RESULTS: We found that even in cases of very rapidly spreading smallpox, ring vaccination (when coupled with surveillance) is sufficient in most cases to eliminate smallpox quickly, assuming that 95% of household contacts are traced, 80% of workplace or social contacts are traced, and no casual contacts are traced, and that in most cases the ability to trace 1–5 individuals per day per index case is sufficient. If smallpox is assumed to be transmitted very quickly to contacts, it may at times escape containment by ring vaccination, but could be controlled in these circumstances by mass vaccination. CONCLUSIONS: Small introductions of smallpox are likely to be easily contained by ring vaccination, provided contact tracing is feasible. Uncertainties in the nature of bioterrorist smallpox (infectiousness, vaccine efficacy) support continued planning for ring vaccination as well as mass vaccination. If initiated, ring vaccination should be conducted without delays in vaccination, should include contacts of contacts (whenever there is sufficient capacity) and should be accompanied by increased public awareness and surveillance.",
         "2004-08-06",
         "Porco, Travis C; Holbrook, Karen A; Fernyak, Susan E; Portnoy, Diane L; Reiter, Randy; Aragón, Tomás J",
         "BMC Public Health",
         null,
         null,
         null,
         "document_parses/pdf_json/367af6bb9a8bbda02206506a819656bb6f14ce4c.json",
         "document_parses/pmc_json/PMC520756.xml.json",
         "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC520756/",
         null
        ]
       ],
       "shape": {
        "columns": 19,
        "rows": 128492
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>cord_uid</th>\n",
       "      <th>sha</th>\n",
       "      <th>source_x</th>\n",
       "      <th>title</th>\n",
       "      <th>doi</th>\n",
       "      <th>pmcid</th>\n",
       "      <th>pubmed_id</th>\n",
       "      <th>license</th>\n",
       "      <th>abstract</th>\n",
       "      <th>publish_time</th>\n",
       "      <th>authors</th>\n",
       "      <th>journal</th>\n",
       "      <th>mag_id</th>\n",
       "      <th>who_covidence_id</th>\n",
       "      <th>arxiv_id</th>\n",
       "      <th>pdf_json_files</th>\n",
       "      <th>pmc_json_files</th>\n",
       "      <th>url</th>\n",
       "      <th>s2_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ug7v899j</td>\n",
       "      <td>d1aafb70c066a2068b02786f8929fd9c900897fb</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Clinical features of culture-proven Mycoplasma...</td>\n",
       "      <td>10.1186/1471-2334-1-6</td>\n",
       "      <td>PMC35282</td>\n",
       "      <td>11472636.0</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>OBJECTIVE: This retrospective chart review des...</td>\n",
       "      <td>2001-07-04</td>\n",
       "      <td>Madani, Tariq A; Al-Ghamdi, Aisha A</td>\n",
       "      <td>BMC Infect Dis</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>document_parses/pdf_json/d1aafb70c066a2068b027...</td>\n",
       "      <td>document_parses/pmc_json/PMC35282.xml.json</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>02tnwd4m</td>\n",
       "      <td>6b0567729c2143a66d737eb0a2f63f2dce2e5a7d</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Nitric oxide: a pro-inflammatory mediator in l...</td>\n",
       "      <td>10.1186/rr14</td>\n",
       "      <td>PMC59543</td>\n",
       "      <td>11667967.0</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>Inflammatory diseases of the respiratory tract...</td>\n",
       "      <td>2000-08-15</td>\n",
       "      <td>Vliet, Albert van der; Eiserich, Jason P; Cros...</td>\n",
       "      <td>Respir Res</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>document_parses/pdf_json/6b0567729c2143a66d737...</td>\n",
       "      <td>document_parses/pmc_json/PMC59543.xml.json</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>ejv2xln0</td>\n",
       "      <td>06ced00a5fc04215949aa72528f2eeaae1d58927</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Surfactant protein-D and pulmonary host defense</td>\n",
       "      <td>10.1186/rr19</td>\n",
       "      <td>PMC59549</td>\n",
       "      <td>11667972.0</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>Surfactant protein-D (SP-D) participates in th...</td>\n",
       "      <td>2000-08-25</td>\n",
       "      <td>Crouch, Erika C</td>\n",
       "      <td>Respir Res</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>document_parses/pdf_json/06ced00a5fc04215949aa...</td>\n",
       "      <td>document_parses/pmc_json/PMC59549.xml.json</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>2b73a28n</td>\n",
       "      <td>348055649b6b8cf2b9a376498df9bf41f7123605</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Role of endothelin-1 in lung disease</td>\n",
       "      <td>10.1186/rr44</td>\n",
       "      <td>PMC59574</td>\n",
       "      <td>11686871.0</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>Endothelin-1 (ET-1) is a 21 amino acid peptide...</td>\n",
       "      <td>2001-02-22</td>\n",
       "      <td>Fagan, Karen A; McMurtry, Ivan F; Rodman, David M</td>\n",
       "      <td>Respir Res</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>document_parses/pdf_json/348055649b6b8cf2b9a37...</td>\n",
       "      <td>document_parses/pmc_json/PMC59574.xml.json</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>9785vg6d</td>\n",
       "      <td>5f48792a5fa08bed9f56016f4981ae2ca6031b32</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Gene expression in epithelial cells in respons...</td>\n",
       "      <td>10.1186/rr61</td>\n",
       "      <td>PMC59580</td>\n",
       "      <td>11686888.0</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>Respiratory syncytial virus (RSV) and pneumoni...</td>\n",
       "      <td>2001-05-11</td>\n",
       "      <td>Domachowske, Joseph B; Bonville, Cynthia A; Ro...</td>\n",
       "      <td>Respir Res</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>document_parses/pdf_json/5f48792a5fa08bed9f560...</td>\n",
       "      <td>document_parses/pmc_json/PMC59580.xml.json</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128487</th>\n",
       "      <td>a11jyui1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Elsevier; PMC</td>\n",
       "      <td>Impact of BSE on the biotechnology industry — ...</td>\n",
       "      <td>10.1016/s0958-2118(04)00238-1</td>\n",
       "      <td>PMC7148827</td>\n",
       "      <td>NaN</td>\n",
       "      <td>els-covid</td>\n",
       "      <td>This feature briefly looks at filtration techn...</td>\n",
       "      <td>2004-10-31</td>\n",
       "      <td>Atkinson, Simon</td>\n",
       "      <td>Membrane Technology</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>109580667.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128488</th>\n",
       "      <td>9bma9y0q</td>\n",
       "      <td>44959e4505dcc59575f4989de8abf3fcc7e48415</td>\n",
       "      <td>Medline; PMC</td>\n",
       "      <td>Evolving Gene Targets and Technology in Influe...</td>\n",
       "      <td>10.1007/s40291-013-0040-9</td>\n",
       "      <td>PMC7100497</td>\n",
       "      <td>23686537.0</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>Influenza viruses cause recurring epidemic out...</td>\n",
       "      <td>2013-05-18</td>\n",
       "      <td>Malanoski, Anthony P.; Lin, Baochuan</td>\n",
       "      <td>Mol Diagn Ther</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>document_parses/pdf_json/44959e4505dcc59575f49...</td>\n",
       "      <td>document_parses/pmc_json/PMC7100497.xml.json</td>\n",
       "      <td>NaN</td>\n",
       "      <td>17361009.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128489</th>\n",
       "      <td>50xhia4r</td>\n",
       "      <td>8922380f61b6fe160ab0550a674816f1852a843d</td>\n",
       "      <td>Medline; PMC; WHO</td>\n",
       "      <td>How to train health personnel to protect thems...</td>\n",
       "      <td>10.3352/jeehp.2020.17.10</td>\n",
       "      <td>PMC7162995</td>\n",
       "      <td>32150796.0</td>\n",
       "      <td>cc-by</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2020-03-07</td>\n",
       "      <td>Huh, Sun</td>\n",
       "      <td>J Educ Eval Health Prof</td>\n",
       "      <td>NaN</td>\n",
       "      <td>#6263</td>\n",
       "      <td>NaN</td>\n",
       "      <td>document_parses/pdf_json/8922380f61b6fe160ab05...</td>\n",
       "      <td>document_parses/pmc_json/PMC7162995.xml.json</td>\n",
       "      <td>NaN</td>\n",
       "      <td>212650605.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128490</th>\n",
       "      <td>97h1fz34</td>\n",
       "      <td>ea20a6b97f98ce4bdddfa6408da5ddf3dd06835e</td>\n",
       "      <td>Medline; PMC</td>\n",
       "      <td>Epidemic spreading in complex networks</td>\n",
       "      <td>10.1007/s11467-008-0027-x</td>\n",
       "      <td>PMC7111544</td>\n",
       "      <td>32288753.0</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>The study of epidemic spreading in complex net...</td>\n",
       "      <td>2008-07-08</td>\n",
       "      <td>Zhou, Jie; Liu, Zong-hua</td>\n",
       "      <td>Front Phys China</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>document_parses/pdf_json/ea20a6b97f98ce4bdddfa...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>121665618.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128491</th>\n",
       "      <td>clmtwq4v</td>\n",
       "      <td>4c779b2a7682e10c421d6df6175226115a703997</td>\n",
       "      <td>Elsevier; Medline; PMC</td>\n",
       "      <td>The determinants of the 1999 and 2007 Chinese ...</td>\n",
       "      <td>10.1016/j.tourman.2009.10.003</td>\n",
       "      <td>PMC7127154</td>\n",
       "      <td>32287734.0</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>This study examines the factors that lay behin...</td>\n",
       "      <td>2009-11-06</td>\n",
       "      <td>York, Qi Yan; Zhang, Hanqin Qiu</td>\n",
       "      <td>Tour Manag</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>document_parses/pdf_json/4c779b2a7682e10c421d6...</td>\n",
       "      <td>document_parses/pmc_json/PMC7127154.xml.json</td>\n",
       "      <td>https://www.sciencedirect.com/science/article/...</td>\n",
       "      <td>154015971.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>128492 rows × 19 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        cord_uid                                       sha  \\\n",
       "0       ug7v899j  d1aafb70c066a2068b02786f8929fd9c900897fb   \n",
       "1       02tnwd4m  6b0567729c2143a66d737eb0a2f63f2dce2e5a7d   \n",
       "2       ejv2xln0  06ced00a5fc04215949aa72528f2eeaae1d58927   \n",
       "3       2b73a28n  348055649b6b8cf2b9a376498df9bf41f7123605   \n",
       "4       9785vg6d  5f48792a5fa08bed9f56016f4981ae2ca6031b32   \n",
       "...          ...                                       ...   \n",
       "128487  a11jyui1                                       NaN   \n",
       "128488  9bma9y0q  44959e4505dcc59575f4989de8abf3fcc7e48415   \n",
       "128489  50xhia4r  8922380f61b6fe160ab0550a674816f1852a843d   \n",
       "128490  97h1fz34  ea20a6b97f98ce4bdddfa6408da5ddf3dd06835e   \n",
       "128491  clmtwq4v  4c779b2a7682e10c421d6df6175226115a703997   \n",
       "\n",
       "                      source_x  \\\n",
       "0                          PMC   \n",
       "1                          PMC   \n",
       "2                          PMC   \n",
       "3                          PMC   \n",
       "4                          PMC   \n",
       "...                        ...   \n",
       "128487           Elsevier; PMC   \n",
       "128488            Medline; PMC   \n",
       "128489       Medline; PMC; WHO   \n",
       "128490            Medline; PMC   \n",
       "128491  Elsevier; Medline; PMC   \n",
       "\n",
       "                                                    title  \\\n",
       "0       Clinical features of culture-proven Mycoplasma...   \n",
       "1       Nitric oxide: a pro-inflammatory mediator in l...   \n",
       "2         Surfactant protein-D and pulmonary host defense   \n",
       "3                    Role of endothelin-1 in lung disease   \n",
       "4       Gene expression in epithelial cells in respons...   \n",
       "...                                                   ...   \n",
       "128487  Impact of BSE on the biotechnology industry — ...   \n",
       "128488  Evolving Gene Targets and Technology in Influe...   \n",
       "128489  How to train health personnel to protect thems...   \n",
       "128490             Epidemic spreading in complex networks   \n",
       "128491  The determinants of the 1999 and 2007 Chinese ...   \n",
       "\n",
       "                                  doi       pmcid   pubmed_id    license  \\\n",
       "0               10.1186/1471-2334-1-6    PMC35282  11472636.0      no-cc   \n",
       "1                        10.1186/rr14    PMC59543  11667967.0      no-cc   \n",
       "2                        10.1186/rr19    PMC59549  11667972.0      no-cc   \n",
       "3                        10.1186/rr44    PMC59574  11686871.0      no-cc   \n",
       "4                        10.1186/rr61    PMC59580  11686888.0      no-cc   \n",
       "...                               ...         ...         ...        ...   \n",
       "128487  10.1016/s0958-2118(04)00238-1  PMC7148827         NaN  els-covid   \n",
       "128488      10.1007/s40291-013-0040-9  PMC7100497  23686537.0      no-cc   \n",
       "128489       10.3352/jeehp.2020.17.10  PMC7162995  32150796.0      cc-by   \n",
       "128490      10.1007/s11467-008-0027-x  PMC7111544  32288753.0      no-cc   \n",
       "128491  10.1016/j.tourman.2009.10.003  PMC7127154  32287734.0      no-cc   \n",
       "\n",
       "                                                 abstract publish_time  \\\n",
       "0       OBJECTIVE: This retrospective chart review des...   2001-07-04   \n",
       "1       Inflammatory diseases of the respiratory tract...   2000-08-15   \n",
       "2       Surfactant protein-D (SP-D) participates in th...   2000-08-25   \n",
       "3       Endothelin-1 (ET-1) is a 21 amino acid peptide...   2001-02-22   \n",
       "4       Respiratory syncytial virus (RSV) and pneumoni...   2001-05-11   \n",
       "...                                                   ...          ...   \n",
       "128487  This feature briefly looks at filtration techn...   2004-10-31   \n",
       "128488  Influenza viruses cause recurring epidemic out...   2013-05-18   \n",
       "128489                                                NaN   2020-03-07   \n",
       "128490  The study of epidemic spreading in complex net...   2008-07-08   \n",
       "128491  This study examines the factors that lay behin...   2009-11-06   \n",
       "\n",
       "                                                  authors  \\\n",
       "0                     Madani, Tariq A; Al-Ghamdi, Aisha A   \n",
       "1       Vliet, Albert van der; Eiserich, Jason P; Cros...   \n",
       "2                                         Crouch, Erika C   \n",
       "3       Fagan, Karen A; McMurtry, Ivan F; Rodman, David M   \n",
       "4       Domachowske, Joseph B; Bonville, Cynthia A; Ro...   \n",
       "...                                                   ...   \n",
       "128487                                    Atkinson, Simon   \n",
       "128488               Malanoski, Anthony P.; Lin, Baochuan   \n",
       "128489                                           Huh, Sun   \n",
       "128490                           Zhou, Jie; Liu, Zong-hua   \n",
       "128491                    York, Qi Yan; Zhang, Hanqin Qiu   \n",
       "\n",
       "                        journal  mag_id who_covidence_id arxiv_id  \\\n",
       "0                BMC Infect Dis     NaN              NaN      NaN   \n",
       "1                    Respir Res     NaN              NaN      NaN   \n",
       "2                    Respir Res     NaN              NaN      NaN   \n",
       "3                    Respir Res     NaN              NaN      NaN   \n",
       "4                    Respir Res     NaN              NaN      NaN   \n",
       "...                         ...     ...              ...      ...   \n",
       "128487      Membrane Technology     NaN              NaN      NaN   \n",
       "128488           Mol Diagn Ther     NaN              NaN      NaN   \n",
       "128489  J Educ Eval Health Prof     NaN            #6263      NaN   \n",
       "128490         Front Phys China     NaN              NaN      NaN   \n",
       "128491               Tour Manag     NaN              NaN      NaN   \n",
       "\n",
       "                                           pdf_json_files  \\\n",
       "0       document_parses/pdf_json/d1aafb70c066a2068b027...   \n",
       "1       document_parses/pdf_json/6b0567729c2143a66d737...   \n",
       "2       document_parses/pdf_json/06ced00a5fc04215949aa...   \n",
       "3       document_parses/pdf_json/348055649b6b8cf2b9a37...   \n",
       "4       document_parses/pdf_json/5f48792a5fa08bed9f560...   \n",
       "...                                                   ...   \n",
       "128487                                                NaN   \n",
       "128488  document_parses/pdf_json/44959e4505dcc59575f49...   \n",
       "128489  document_parses/pdf_json/8922380f61b6fe160ab05...   \n",
       "128490  document_parses/pdf_json/ea20a6b97f98ce4bdddfa...   \n",
       "128491  document_parses/pdf_json/4c779b2a7682e10c421d6...   \n",
       "\n",
       "                                      pmc_json_files  \\\n",
       "0         document_parses/pmc_json/PMC35282.xml.json   \n",
       "1         document_parses/pmc_json/PMC59543.xml.json   \n",
       "2         document_parses/pmc_json/PMC59549.xml.json   \n",
       "3         document_parses/pmc_json/PMC59574.xml.json   \n",
       "4         document_parses/pmc_json/PMC59580.xml.json   \n",
       "...                                              ...   \n",
       "128487                                           NaN   \n",
       "128488  document_parses/pmc_json/PMC7100497.xml.json   \n",
       "128489  document_parses/pmc_json/PMC7162995.xml.json   \n",
       "128490                                           NaN   \n",
       "128491  document_parses/pmc_json/PMC7127154.xml.json   \n",
       "\n",
       "                                                      url        s2_id  \n",
       "0       https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3...          NaN  \n",
       "1       https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...          NaN  \n",
       "2       https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...          NaN  \n",
       "3       https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...          NaN  \n",
       "4       https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...          NaN  \n",
       "...                                                   ...          ...  \n",
       "128487  https://www.sciencedirect.com/science/article/...  109580667.0  \n",
       "128488                                                NaN   17361009.0  \n",
       "128489                                                NaN  212650605.0  \n",
       "128490                                                NaN  121665618.0  \n",
       "128491  https://www.sciencedirect.com/science/article/...  154015971.0  \n",
       "\n",
       "[128492 rows x 19 columns]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = pd.read_csv(os.path.join(__root__, '..', 'data', 'raw', 'CORD_19', 'metadata.csv'))\n",
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e5628f47",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "myenv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
